# translation of ja.po to Japanese
# anaconda/ja.po
# Copyright (C) 2001 Red Hat, Inc.
# SATO Satoru <ssato@redhat.com>
#
# James Hashida <khashida@redhat.com>, 2002,2003.
# Noriko Mizumoto <noriko@redhat.com>, 2003, 2004, 2006, 2008, 2009.
# Yoshinari Takaoka <mumumu@mx.edit.ne.jp>, 2004.
# Tadashi Jokagi <fedora@elf.no-ip.org>, 2004, 2005.
# Naoki Shigematsu <sigematu@linuxml.net>, 2004.
# Hirofumi Saito <hi_saito@yk.rim.or.jp>, 2004, 2005, 2006.
# Kiyoto James Hashida <khashida@redhat.com>, 2006, 2007.
# Hyu_gabaru Ryu_ichi <hyu_gabaru@yahoo.co.jp>, 2007, 2009.
# Kiyoto Hashida <khashida@redhat.com>, 2007, 2008, 2009, 2010.
# Makoto Mizukami <mmk9060@aol.com>, 2008.
msgid ""
msgstr ""
"Project-Id-Version: ja\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2009-11-06 13:53-1000\n"
"PO-Revision-Date: 2010-04-26 14:02+0900\n"
"Last-Translator: Kiyoto Hashida <khashida@redhat.com>\n"
"Language-Team: Japanese <jp@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: KBabel 1.11.4\n"
"Plural-Forms: Plural-Forms: nplurals=1; plural=0;\n"
"\n"
"\n"
"\n"

#: anaconda:361
msgid "Press <enter> for a shell"
msgstr "シェルには、<enter> を押します。"

#: anaconda:376 rescue.py:324 rescue.py:350 rescue.py:363 rescue.py:439
#: rescue.py:451 text.py:500 loader/cdinstall.c:228 loader/cdinstall.c:232
#: loader/cdinstall.c:248 loader/cdinstall.c:251 loader/cdinstall.c:445
#: loader/cdinstall.c:452 loader/cdinstall.c:517 loader/dirbrowser.c:145
#: loader/driverdisk.c:98 loader/driverdisk.c:243 loader/driverdisk.c:274
#: loader/driverdisk.c:305 loader/driverdisk.c:340 loader/driverdisk.c:359
#: loader/driverdisk.c:371 loader/driverdisk.c:379 loader/driverdisk.c:539
#: loader/driverdisk.c:576 loader/driverselect.c:81 loader/driverselect.c:205
#: loader/hdinstall.c:134 loader/hdinstall.c:279 loader/hdinstall.c:340
#: loader/hdinstall.c:372 loader/hdinstall.c:442 loader/hdinstall.c:485
#: loader/hdinstall.c:498 loader/kbd.c:119 loader/kickstart.c:132
#: loader/kickstart.c:142 loader/kickstart.c:185 loader/kickstart.c:284
#: loader/kickstart.c:382 loader/kickstart.c:563 loader/lang.c:115
#: loader/lang.c:379 loader/loader.c:403 loader/loader.c:439
#: loader/loader.c:482 loader/loader.c:503 loader/loader.c:520
#: loader/loader.c:557 loader/loader.c:1191 loader/loader.c:1375
#: loader/mediacheck.c:46 loader/mediacheck.c:85 loader/mediacheck.c:92
#: loader/mediacheck.c:101 loader/method.c:279 loader/method.c:353
#: loader/modules.c:364 loader/modules.c:376 loader/net.c:582 loader/net.c:953
#: loader/net.c:1587 loader/net.c:1608 loader/net.c:1880 loader/net.c:1902
#: loader/net.c:1914 loader/nfsinstall.c:91 loader/nfsinstall.c:250
#: loader/nfsinstall.c:269 loader/nfsinstall.c:337 loader/telnetd.c:92
#: loader/urlinstall.c:83 loader/urlinstall.c:107 loader/urlinstall.c:175
#: loader/urlinstall.c:451 loader/urlinstall.c:460 loader/urlinstall.c:467
#: loader/urls.c:292 loader/urls.c:328 loader/urls.c:334 loader/urls.c:346
#: textw/constants_text.py:44
msgid "OK"
msgstr "OK"

#: anaconda:383
msgid ""
"You do not have enough RAM to use the graphical installer.  Starting text "
"mode."
msgstr ""
"グラフィカルインストーラーを使用するにはメモリが足りません。テキストモードで"
"起動します"

#: anaconda:446
msgid "Starting graphical installation."
msgstr "グラフィカルインストールを開始。"

#: anaconda:762
msgid "Would you like to use VNC?"
msgstr "VNC を使用しますか ?"

#: anaconda:763
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""
"テキストモードは限定的なインストールオプションのセットを提供します。好みの "
"パーティション設定やパッケージ選択の指定がありません。代わりに VNC モードを "
"使用しますか？"

#: anaconda:789
msgid "Install class forcing text mode installation"
msgstr "テキストモードインストールを強制するインストールクラス"

#: anaconda:808
msgid "Graphical installation is not available. Starting text mode."
msgstr "グラフィカルインストールは使用できません。テキストモードを起動します。"

#: anaconda:816
msgid "DISPLAY variable not set. Starting text mode."
msgstr "DISPLAY 変数がセットされていません。テキストモードで起動します。"

#: anaconda:984
msgid "reipl configuration successful => reboot"
msgstr ""

#: anaconda:987
msgid "reipl configuration failed => halt"
msgstr ""

#: backend.py:148
#, python-format
msgid "Upgrading %s\n"
msgstr "%s のアップグレード\n"

#: backend.py:150
#, python-format
msgid "Installing %s\n"
msgstr "%s のインストール\n"

#: backend.py:170
msgid "Copying File"
msgstr "ファイルをコピー中"

#: backend.py:171
msgid "Transferring install image to hard drive"
msgstr "インストールイメージをハードドライブに転送中"

#: backend.py:181
msgid ""
"An error occurred transferring the install image to your hard drive.  This "
"is often cause by damaged or low quality media."
msgstr ""
"インストールイメージをハードドライブへ転送している際にエラーが発生しました。"
"これは傷んだ、または品質の悪い媒体が原因でよく発生します。"

#: backend.py:185
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""
"インストールイメージをハードドライブへ転送中にエラーが発生しました。おそらく"
"ディスク容量が不足しています。"

#: backend.py:189 image.py:285 livecd.py:190 livecd.py:396
#: partIntfHelpers.py:237 text.py:308 text.py:312 yuminstall.py:394
#: yuminstall.py:734 yuminstall.py:848 yuminstall.py:1111 yuminstall.py:1175
#: yuminstall.py:1334 yuminstall.py:1352 iw/autopart_type.py:99
#: iw/autopart_type.py:198 iw/autopart_type.py:325 iw/autopart_type.py:328
#: iw/autopart_type.py:395 iw/autopart_type.py:403 iw/autopart_type.py:432
#: iw/osbootwidget.py:210 iw/osbootwidget.py:219 iw/raid_dialog_gui.py:724
#: iw/raid_dialog_gui.py:763 iw/task_gui.py:63 iw/task_gui.py:164
#: iw/task_gui.py:307 iw/task_gui.py:442 loader/cdinstall.c:228
#: loader/cdinstall.c:517 loader/driverdisk.c:98 loader/driverdisk.c:305
#: loader/driverdisk.c:340 loader/driverdisk.c:371 loader/driverdisk.c:379
#: loader/driverdisk.c:443 loader/hdinstall.c:134 loader/hdinstall.c:340
#: loader/hdinstall.c:442 loader/hdinstall.c:485 loader/hdinstall.c:498
#: loader/kickstart.c:284 loader/lang.c:115 loader/loader.c:403
#: loader/loader.c:520 loader/loader.c:1191 loader/mediacheck.c:46
#: loader/mediacheck.c:85 loader/mediacheck.c:92 loader/method.c:279
#: loader/method.c:353 loader/nfsinstall.c:250 loader/nfsinstall.c:269
#: loader/telnetd.c:92 loader/urlinstall.c:83 loader/urlinstall.c:107
#: loader/urlinstall.c:175 loader/urls.c:328 loader/urls.c:334
#: storage/__init__.py:200 storage/__init__.py:1645 storage/__init__.py:1730
#: textw/netconfig_text.py:280 textw/partition_text.py:186
#: textw/partition_text.py:192 textw/partition_text.py:198
#: textw/partition_text.py:231 textw/partition_text.py:241
#: textw/upgrade_text.py:175 textw/upgrade_text.py:182
msgid "Error"
msgstr "エラー"

#: bootloader.py:53 bootloader.py:208 bootloader.py:215 image.py:82
#: upgrade.py:251 yuminstall.py:1515 yuminstall.py:1551
#: iw/autopart_type.py:208 iw/blpasswidget.py:148 iw/upgrade_swap_gui.py:191
#: iw/upgrade_swap_gui.py:199 iw/upgrade_swap_gui.py:206
#: storage/devicetree.py:145 storage/devicetree.py:172
#: textw/upgrade_text.py:187
msgid "Warning"
msgstr "警告"

#: bootloader.py:54
msgid ""
"Filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""
"ファイルシステムは既に活性化しています。ここから元に戻ることはできません。\n"
"\n"
"インストールを続けますか?"

#: bootloader.py:58 gui.py:1097 gui.py:1134 gui.py:1262 gui.py:1341
#: image.py:91 kickstart.py:151 livecd.py:197 livecd.py:403 packages.py:119
#: packages.py:127 packages.py:135 packages.py:143 packages.py:155
#: packages.py:164 packages.py:172 packages.py:181 upgrade.py:58
#: upgrade.py:206 yuminstall.py:236 yuminstall.py:665 yuminstall.py:852
#: yuminstall.py:932 yuminstall.py:938 yuminstall.py:1082 yuminstall.py:1107
#: yuminstall.py:1165 yuminstall.py:1329 yuminstall.py:1359
#: storage/__init__.py:96 storage/__init__.py:204 storage/__init__.py:1818
msgid "_Exit installer"
msgstr "インストーラを終了する(_E)"

#: bootloader.py:58 image.py:92 image.py:248 kickstart.py:1357
#: kickstart.py:1396 upgrade.py:58 yuminstall.py:1170 iw/partition_gui.py:1038
#: storage/__init__.py:1819
msgid "_Continue"
msgstr "続行(_C)"

#: bootloader.py:142
msgid "Bootloader"
msgstr "ブートローダー"

#: bootloader.py:142
msgid "Installing bootloader."
msgstr "ブートローダーをインストール中です。"

#: bootloader.py:209
msgid ""
"There was an error installing the bootloader.  The system may not be "
"bootable."
msgstr ""
"ブートローダーのインストールでエラーが発生しました。システムは起動できないか"
"もしれません。"

#: bootloader.py:216
msgid ""
"No kernel packages were installed on the system.  Bootloader configuration "
"will not be changed."
msgstr ""
"システムにカーネルパッケージはインストールされませんでした。ブートローダー設"
"定は変更されません。"

#: cmdline.py:55
msgid "Completed"
msgstr "完了"

#: cmdline.py:63
msgid "In progress"
msgstr "進行中"

#: cmdline.py:86 gui.py:1129 kickstart.py:1237 text.py:345
#, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""
"キックスタート設定ファイルの解析中に次のエラーが見つかりました。\n"
"\n"
"%s"

#: cmdline.py:98
msgid ""
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""
"コマンドラインモードでは kickstart 設定ファイルに全ての選択が設定されている必"
"要があります。"

#: cmdline.py:114 cmdline.py:121 cmdline.py:128
msgid "Can't have a question in command line mode!"
msgstr "コマンドラインモードでは質問は置けません!"

#: constants.py:74
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""
"未処理の例外が発生しました。バグの可能性が考えられます。全てのメッセージをコ"
"ピーし、バグとして報告してください。"

#: constants.py:80
msgid " with the provider of this software."
msgstr "このソフトウェアの供給者と共に"

#: constants.py:84
#, python-format
msgid " against anaconda at %s"
msgstr "anaconda の %s に対して"

#: gui.py:106
msgid "An error occurred saving screenshots to disk."
msgstr "スクリーンショットをディスクに保存中にエラーが発生しました。"

#: gui.py:117
msgid "Screenshots Copied"
msgstr "スクリーンショットがコピーされました"

#: gui.py:118
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""
"スクリーンショットは次のディレクトリーに保存されました:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"再起動して root でログインするとアクセスできます。"

#: gui.py:161
msgid "Saving Screenshot"
msgstr "スクリーンショットを保存中"

#: gui.py:162
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "「%s」 と言う名前のスクリーンショットが保存されました。"

#: gui.py:165
msgid "Error Saving Screenshot"
msgstr "スクリーンショット保存中にエラー発生"

#: gui.py:166
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""
"スクリーンショット保存中にエラーが発生しました。パッケージインストール中に発"
"生した場合、完了するには数回試行する必要があるかもしれません。"

#: gui.py:560
msgid "Installation Key"
msgstr "インストールキー"

#: gui.py:631 text.py:148
msgid ""
"Choose a passphrase for the encrypted devices. You will be prompted for this "
"passphrase during system boot."
msgstr ""
"暗号化されたデバイス用のパスフレーズを選択してください。システムをブートする"
"時にそのパスフレーズを求められます。"

#: gui.py:650 gui.py:658 text.py:186 text.py:196
msgid "Error with passphrase"
msgstr "パスフレーズが違います。"

#: gui.py:651 text.py:187
msgid "The passphrases you entered were different.  Please try again."
msgstr "入力したパスフレーズが異ります。やり直してください。"

#: gui.py:659
msgid "The passphrase must be at least eight characters long."
msgstr "パスフレーズは最低 6 文字以上でなければなりません。"

#: gui.py:693 text.py:224
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""
"デバイス %s は暗号化されています。インストール中にこのデバイスの内容にアクセ"
"スするには、以下にデバイスのパスフレーズを入力する必要があります。"

#: gui.py:784 gui.py:1341 partIntfHelpers.py:157 partIntfHelpers.py:339
#: text.py:99 text.py:100 iw/account_gui.py:130 loader/dirbrowser.c:145
#: loader/driverdisk.c:244 loader/kickstart.c:382 loader/loader.c:439
#: loader/loader.c:557 textw/constants_text.py:48
msgid "Cancel"
msgstr "取り消し"

#: gui.py:796 gui.py:797 gui.py:913 gui.py:914 ui/anaconda.glade.h:3
msgid "_Debug"
msgstr "デバッグ(_D)"

#: gui.py:1131 text.py:347
msgid "Error Parsing Kickstart Config"
msgstr "キックスタート設定ファイル解析エラー"

#: gui.py:1171
msgid "default:LTR"
msgstr "デフォルト:LTR"

#: gui.py:1251 text.py:465
msgid "Error!"
msgstr "エラー!"

#: gui.py:1252 text.py:466
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""
"インストーラーインターフェースコンポーネントを読み込もうとしてエラーが発生し"
"ました。\n"
"\n"
"className = %s"

#: gui.py:1257 image.py:164 image.py:194 packages.py:376 packages.py:381
#: storage/__init__.py:1644
msgid "_Exit"
msgstr "終了(_E)"

#: gui.py:1258 image.py:164 image.py:194 livecd.py:197 yuminstall.py:726
#: yuminstall.py:1107 yuminstall.py:1165 yuminstall.py:1329
msgid "_Retry"
msgstr "再試行(_R)"

#: gui.py:1261
#, fuzzy
msgid "The installer will now exit."
msgstr "システムが再起動します。"

#: gui.py:1264 storage/partitioning.py:267
msgid "The system will now reboot."
msgstr "システムが再起動します。"

#: gui.py:1265 image.py:248 packages.py:384 yuminstall.py:1375
msgid "_Reboot"
msgstr "再起動(_R)"

#: gui.py:1267
msgid "Exiting"
msgstr "終了"

#: gui.py:1338 livecd.py:115 text.py:266 upgrade.py:197
msgid "Exit installer"
msgstr "インストーラーを終了する"

#: gui.py:1339
msgid "Are you sure you wish to exit the installer?"
msgstr "本当にインストーラーを終了しますか?"

#: gui.py:1348
#, python-format
msgid "%s Installer"
msgstr "%s インストーラー"

#: gui.py:1354
msgid "Unable to load title bar"
msgstr "タイトルバーを読み込めません"

#: gui.py:1416
msgid "Install Window"
msgstr "インストールウィンドウ"

#: image.py:83
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""
"この ISO イメージ %s のサイズは、2048 バイトの倍数ではありません。これは、こ"
"のコンピューターへの転送の際に、それが破壊されたことを意味します。\n"
"\n"
"終了して、インストールを中止することをお勧めしますが、もしこれが間違いだと思"
"う場合は、続行することができます。 "

#: image.py:156
msgid "Couldn't Mount ISO Source"
msgstr "ISO ソースをマウントできない"

#: image.py:157
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""
"ソースデバイス %s のマウントでエラーが起きました。LVM や RAID のような進化し"
"たストレージデバイスに ISO イメージが置かれたときや、パーティションのマウント"
"に問題がある場合に発生します。「終了」をクリックしてインストールを中止してく"
"ださい。"

#: image.py:185
msgid "Missing ISO 9660 Image"
msgstr "ISO 9660 イメージがありません"

#: image.py:186
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""
"インストーラーはイメージ #%s のマウントを試行しましたが、ハードドライブ上に見"
"つかりませんでした。\n"
"\n"
"このイメージをドライブにコピーしてから「再試行」をクリックしてください。「終"
"了」をクリックするとインストールを中止します。"

#: image.py:241
msgid "Required Install Media"
msgstr "必要なインストールメディア"

#: image.py:242
#, fuzzy, python-format
msgid ""
"The software you have selected to install will require the following %s %s "
"discs:\n"
"\n"
"%s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""
"インストール用に選択したソフトウェアには次の %(productName)s  %"
"(productVersion)s ディスクが 必要です:\n"
"\n"
"%(reqcdstr)s\n"
"インストールを続行する前にこれらのディスクを用意してください。インストールを"
"中止して操作を終了する必要がある場合は「再起動」を選択します。"

#: image.py:248 livecd.py:402 packages.py:381 packages.py:384 upgrade.py:205
#: yuminstall.py:852 yuminstall.py:938 yuminstall.py:1359 yuminstall.py:1375
#: ui/anaconda.glade.h:2
msgid "_Back"
msgstr "戻る(_B)"

#: image.py:286
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""
"ディスクをアンマウント中にエラーが発生しました。tty2 のシェルから %s にアクセ"
"スしていないことを確認してから [OK] をクリックして再試行してください。"

#: installclass.py:73
msgid "Install on System"
msgstr "システム上にインストール"

#: iutil.py:831
#, fuzzy, python-format
msgid "Error: On open, cannot set reIPL method to %s (%s: %s)"
msgstr ""
"エラー: 開く時に、reIPL メソッドを %(reipl_type)s (%(filename)s: %(e)s) に"
"セットできません"

#: iutil.py:839
#, fuzzy, python-format
msgid "Error: On write, cannot set reIPL method to %s (%s: %s)"
msgstr ""
"エラー: 書き込みで、reIPL メソッドを %(reipl_type)s (%(filename)s: %(e)s) に"
"セットできません"

#: iutil.py:846
#, fuzzy, python-format
msgid "Error: On close, cannot set reIPL method to %s (%s: %s)"
msgstr ""
"エラー: 閉じる時、reIPL メソッドを %(reipl_type)s (%(filename)s: %(e)s) に"
"セットできません"

#: iutil.py:863
#, fuzzy, python-format
msgid "Error: Could not set %s as reIPL device (%s)"
msgstr ""
"エラー: %(device)s を reIPL デバイス (%(e)s) としてセットできませんでした"

#: iutil.py:872
#, python-format
msgid "Error: Could not reset loadparm (%s)"
msgstr "エラー: loadparm (%s) をリセットできません"

#: iutil.py:881
#, python-format
msgid "Warning: Could not reset parm (%s)"
msgstr "警告: parm (%s) をリセットできません"

#: iutil.py:891
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from DASD device %s to continue "
"installation"
msgstr ""
"シャットダウンの後に、DASD デバイス %s から手動で IPL を実行してインストール"
"を 継続してください"

#: iutil.py:913
#, fuzzy, python-format
msgid "Error: reading FCP property %s for reIPL (%s)"
msgstr "エラー: reIPL (%(e)s) の FCP プロパティ %(syspath_property)s 読み込み"

#: iutil.py:926
#, fuzzy, python-format
msgid "Error: writing FCP property %s for reIPL (%s)"
msgstr "エラー: reIPL (%(e)s) の FCP プロパティ %(reipl_property)s の書き込み"

#: iutil.py:939
#, fuzzy, python-format
msgid "Error: writing default FCP property %s for reIPL (%s)"
msgstr ""
"エラー: reIPL (%(e)s) のデフォルト FCP プロパティ %(reipl_property)s の書き込"
"み"

#: iutil.py:949
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from FCP %(device)s with WWPN %"
"(wwpn)s and LUN %(lun)s to continue installation"
msgstr ""
"シャットダウンの後に、WWPN %(wwpn)s と LUN %(lun)sを使用して FCP %(device)s "
"から手動で IPL を実行して、インストールを継続して下さい"

#: iutil.py:956
msgid ""
"After shutdown, please perform a manual IPL from the device now containing /"
"boot to continue installation"
msgstr ""
"シャットダウンの後に、今 /boot を収納しているデバイスから手動で IPL を実行し"
"て、 インストールを継続して下さい"

#: iutil.py:964
#, fuzzy
msgid "Error determining mount point type"
msgstr "ブートデバイスのディスク名の決定でエラー"

#: iutil.py:968
msgid "The mount point /boot or / is on a disk that we are not familiar with"
msgstr "マウントポイント /boot 又は / が熟知していないディスク上にあります"

#: kickstart.py:97
#, fuzzy, python-format
msgid ""
"There was an error running the kickstart script at line %s.  You may examine "
"the output in %s.  This is a fatal error and installation will be aborted.  "
"Press the OK button to exit the installer."
msgstr ""
"キックスタートスクリプトの %(lineno)s 行目の実行中にエラーが発生しました。%"
"(msgs)s 内の出力を調査した方が良いでしょう。これは致命的なエラーであり、イン"
"ストールは中止されます。OK ボタンを押すとインストーラーを終了します。"

#: kickstart.py:112 kickstart.py:114
msgid "Scriptlet Failure"
msgstr "Scriptlet の失敗"

#: kickstart.py:145 rescue.py:240 yuminstall.py:660 yuminstall.py:1077
msgid "No Network Available"
msgstr "ネットワークは使用できません。"

#: kickstart.py:146
msgid ""
"Encryption key escrow requires networking, but there was an error enabling "
"the network on your system."
msgstr ""
"暗号化キー escrow は はネットワークを必要とします。しかし使用中のシステムで"
"ネットワークの有効化にエラーが発生しました。"

#: kickstart.py:1223
#, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr "%%ksappend 行目を処理中にエラーが発生しました: %s"

#: kickstart.py:1226
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr "%%ksappend 行目を処理中に不明なエラーが発生しました: %s"

#: kickstart.py:1298 livecd.py:221
msgid "Post-Installation"
msgstr "インストール後"

#: kickstart.py:1299
msgid "Running post-installation scripts"
msgstr "インストール後スクリプトを実行中"

#: kickstart.py:1315
msgid "Pre-Installation"
msgstr "インストール前"

#: kickstart.py:1316
msgid "Running pre-installation scripts"
msgstr "インストール前スクリプトを実行中"

#: kickstart.py:1348
msgid "Missing Package"
msgstr "見つからないパッケージ"

#: kickstart.py:1349
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort this installation?"
msgstr ""
"パッケージ '%s'をインストールするように指定しています。このパッケージは存在し"
"ません。インストールを続行しますか、中止しますか?"

#: kickstart.py:1355 kickstart.py:1394
msgid "_Abort"
msgstr "中止(_A)"

#: kickstart.py:1356 kickstart.py:1395
msgid "_Ignore All"
msgstr "全てを無視 (_I)"

#: kickstart.py:1386
msgid "Missing Group"
msgstr "見つからないグループ"

#: kickstart.py:1387
#, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort this installation?"
msgstr ""
"グループ '%s'をインストールするように指定しています。このグループは存在しませ"
"ん。インストールを継続しますか、中止しますか?"

#: kickstart.py:1492
#, python-format
msgid ""
"The kickstart configuration file is missing required information that "
"anaconda cannot prompt for.  Please add the following sections and try "
"again:\n"
"%s"
msgstr ""
"使用中のキックスタートファイルは、アナコンダが催促できない必要情報を欠いてお"
"ります。以下のセクションを追加して再試行して下さい:\n"
"%s"

#: livecd.py:110
msgid "Unable to find image"
msgstr "イメージを検出できません。"

#: livecd.py:111
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""
"与えられた場所は、インストールソースとして使用するための有効な %s ライブ ディ"
"スク ではありません。"

#: livecd.py:172
msgid "Copying live image to hard drive."
msgstr "ライブイメージをハードドライブに転送中。"

#: livecd.py:191
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""
"ライブイメージをハードドライブにインストールする時にエラーが発生しました。 不"
"正なメディアが原因かもしれません。インストールメディアを確認して下さい。\n"
"\n"
"終了するとシステムは矛盾した状態となり、再インストールを必要とするように なり"
"ます。"

#: livecd.py:222
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes."
msgstr ""
"インストール後のファイルシステムの変更を開始しています。これには、数分かかる"
"ことがあります。"

#: livecd.py:397
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""
"作成したルートファイルシステムのサイズが、このライブイメージが必要としている"
"サイズより小さすぎます(最低 %.2f MB は必要です)。"

#: network.py:56
msgid "Hostname must be 255 or fewer characters in length."
msgstr "ホスト名は 255 文字かそれ以下の長さにする必要があります。"

#: network.py:62
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or '0-"
"9'"
msgstr ""
"ホスト名は 「a-z」 又は 「A-Z」又は 「0-9」 の範囲の有効な文字で開始しなけれ"
"ば なりません。"

#: network.py:67
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr ""
"ホスト名には 「a-z」、 「A-Z」、 「0-9」、「-」、 「.」以外の文字は使用できま"
"せん。"

#: network.py:176
msgid "IP address is missing."
msgstr "IP アドレスがありません。"

#: network.py:180
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""
"IPv4 アドレスは、ピリオドで区切られた 0 から 255 までの 4 つの数字を含まなけ"
"ればなりません。"

#: network.py:183
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "%s は有効な IPv6 アドレスではありません。"

#: network.py:185
#, python-format
msgid "'%s' is an invalid IP address."
msgstr "'%s' は、有効な IP アドレスではありません。"

#: packages.py:114
msgid "Device Resize Failed"
msgstr "デバイスのリサイズに失敗"

#: packages.py:115
#, python-format
msgid "An error was encountered while resizing device %s."
msgstr "デバイス %s のリサイズ中にエラーが発生しました。"

#: packages.py:122
msgid "Device Creation Failed"
msgstr "デバイスの作成に失敗"

#: packages.py:123
#, python-format
msgid "An error was encountered while creating device %s."
msgstr "デバイス %s の作成中にエラーが発生しました。"

#: packages.py:130
msgid "Device Removal Failed"
msgstr "デバイスの取り出しに失敗 "

#: packages.py:131
#, python-format
msgid "An error was encountered while removing device %s."
msgstr "デバイス %s の削除中にエラーが発生しました。"

#: packages.py:138
msgid "Device Setup Failed"
msgstr "デバイスのセットアップに失敗"

#: packages.py:139
#, python-format
msgid "An error was encountered while setting up device %s."
msgstr "デバイス %s のセットアップ中にエラーが発生しました。"

#: packages.py:150
msgid "Resizing Failed"
msgstr "リサイズに失敗"

#: packages.py:151
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr "デバイス %s のリサイズ中にエラーが発生しました。"

#: packages.py:158
msgid "Migration Failed"
msgstr "移行に失敗しました。"

#: packages.py:159
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr "デバイス %s 上のファイルシステムを移行中にエラーが発生しました。"

#: packages.py:167
msgid "Formatting Failed"
msgstr "フォーマットが失敗"

#: packages.py:168
#, python-format
msgid "An error was encountered while formatting device %s."
msgstr "デバイス %s のフォーマット中にエラーが発生しました。"

#: packages.py:176
msgid "Storage Activation Failed"
msgstr "ストレージアクティベーションに失敗"

#: packages.py:177
msgid "An error was encountered while activating your storage configuration."
msgstr "使用ストレージの設定をアクティベート中にエラーが発生しました。"

#: packages.py:181
msgid "_File Bug"
msgstr "バグを記録する (_F)"

#: packages.py:362 packages.py:385
msgid "Warning! This is pre-release software!"
msgstr "警告! これはベータ版のソフトウェアです!"

#: packages.py:363
#, fuzzy, python-format
msgid ""
"Thank you for downloading this pre-release of %s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %s\n"
"\n"
"and file a report against '%s'.\n"
msgstr ""
"%(productName)s のプレリリースをダウンロードして頂きありがとうございます。\n"
"\n"
"これは最終リリースではありません。また実稼働用システムでの使用を目的としたも"
"のではありません。このリリースの目的は、テスターからのフィードバックを収集す"
"ることであり、日常の使用には適しません。\n"
"\n"
"フィードバックの報告には以下を開いて下さい:\n"
"\n"
"    %(bugzillaUrl)s\n"
"\n"
"そして %(fileagainst)s に対して報告を提出してください。\n"

#: packages.py:376
msgid "_Install anyway"
msgstr "とにかくインストールする(_I)"

#: packages.py:380
#, fuzzy
msgid "The installer will now exit..."
msgstr "システムが再起動します。"

#: packages.py:383
msgid "Your system will now be rebooted..."
msgstr "システムが再起動します..."

#: partIntfHelpers.py:41
msgid "Please enter a volume group name."
msgstr "ボリュームグループ名を入力してください。"

#: partIntfHelpers.py:45
msgid "Volume Group Names must be less than 128 characters"
msgstr "ボリュームグループ名は 128 文字以下にしてください"

#: partIntfHelpers.py:48
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr "エラー - ボリュームグループ名 %s は有効ではありません。"

#: partIntfHelpers.py:53
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"エラー - ボリュームグループ名に不適切な文字または空白が含まれています。ボ"
"リュームグループ名として使用できる符号は文字、数字、「.」、「_」になります。"

#: partIntfHelpers.py:63
msgid "Please enter a logical volume name."
msgstr "論理ボリューム名を入力してください。"

#: partIntfHelpers.py:67
msgid "Logical Volume Names must be less than 128 characters"
msgstr "論理ボリューム名は 128 文字以下にしてください"

#: partIntfHelpers.py:71
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr "エラー - 論理ボリューム名 %s は有効ではありません。"

#: partIntfHelpers.py:77
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"エラー - 論理ボリューム名に不適切な文字または空白が含まれています。論理ボ"
"リューム名に使用できる符号は文字、数字、「.」、「_」になります。"

#: partIntfHelpers.py:101
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""
"マウントポイント %s は無効です。マウントポイントは 「/」 で始まらなければなら"
"ず、「/」 で終ることはできません。また、空白がなく表示可能な文字で構成されて"
"いなければなりません。"

#: partIntfHelpers.py:108
msgid "Please specify a mount point for this partition."
msgstr "このパーティションに対するマウントポイントを指定してください。"

#: partIntfHelpers.py:121 partIntfHelpers.py:128
msgid "Unable To Delete"
msgstr "削除できません"

#: partIntfHelpers.py:122
msgid "You must first select a partition to delete."
msgstr "まず削除するパーティションを選択してください。"

#: partIntfHelpers.py:153 partIntfHelpers.py:338 iw/lvm_dialog_gui.py:838
msgid "Confirm Delete"
msgstr "削除の確認"

#: partIntfHelpers.py:154
#, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr "デバイス %s 上のすべてのパーティションを削除しようとしています。"

#: partIntfHelpers.py:157 partIntfHelpers.py:339 iw/lvm_dialog_gui.py:841
#: iw/lvm_dialog_gui.py:1370 iw/osbootwidget.py:103 iw/partition_gui.py:1348
msgid "_Delete"
msgstr "削除(_D)"

#: partIntfHelpers.py:204
msgid "Notice"
msgstr "通知"

#: partIntfHelpers.py:205
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"次のパーティションは使用中のため削除されませんでした。\n"
"\n"
"%s"

#: partIntfHelpers.py:220
msgid "Format as Swap?"
msgstr "swap としてフォーマットしますか?"

#: partIntfHelpers.py:221
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""
"%s にはタイプ 0x82 (Linux swap) のパーティションがありますが、Linux swap パー"
"ティションとしてフォーマットされるようには表示されていません。\n"
"\n"
"このパーティションを swap パーティションとしてフォーマットしますか?"

#: partIntfHelpers.py:236
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr ""
"%s をインストールするには、少なくともハードドライブを 1 つ選択しなければなり"
"ません。"

#: partIntfHelpers.py:241
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""
"既存のパーティションをフォーマットせずに、インストールに使用するよう選択して"
"います。前のオペレーティングシステムのファイル群が原因でこの Linux のインス"
"トールに支障をきたさないよう、パーティションのフォーマットを行うことを推奨し"
"ます。尚、ユーザーのホームディレクトリーなど保持しておく必要があるファイルを"
"ある場合は、パーティションのフォーマットを行わずに続行します。"

#: partIntfHelpers.py:249
msgid "Format?"
msgstr "フォーマットしますか?"

#: partIntfHelpers.py:249 iw/partition_gui.py:1036
msgid "_Modify Partition"
msgstr "パーティションの変更(_M)"

#: partIntfHelpers.py:249
msgid "Do _Not Format"
msgstr "フォーマットをしない(_N)"

#: partIntfHelpers.py:257
msgid "Error with Partitioning"
msgstr "パーティション設定エラー"

#: partIntfHelpers.py:258
#, fuzzy, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of %s.\n"
"\n"
"%s"
msgstr ""
"要求されたパーティション設定に次のような致命的なエラーが存在しています。%"
"(productName)s のインストールを続行する前に、これらのエラーを修正する必要があ"
"ります。\n"
"\n"
"%(errorstr)s"

#: partIntfHelpers.py:272
msgid "Partitioning Warning"
msgstr "パーティション設定の警告"

#: partIntfHelpers.py:273
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"要求されたパーティション設定について次のような警告があります。\n"
"\n"
"%s\n"
"\n"
"このパーティション設定のまま続行しますか?"

#: partIntfHelpers.py:287 iw/partition_gui.py:640
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""
"次の既存のパーティションをフォーマットするよう選択されています。これによりす"
"べてのデータが消去されます。"

#: partIntfHelpers.py:290
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""
"処理を続行してこれらのパーティションをフォーマットするには、「はい」を選択し"
"ます。前に戻り設定を変更するには「いいえ」を選択します。"

#: partIntfHelpers.py:296
msgid "Format Warning"
msgstr "フォーマット警告"

#: partIntfHelpers.py:322
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""
"ボリュームグループ \"%s\" を削除しようとしています。\n"
" \n"
"このボリュームグループ内のすべての論理ボリュームが消去されます!"

#: partIntfHelpers.py:326
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr "論理ボリューム \"%s\" を削除しようとしています。"

#: partIntfHelpers.py:329
msgid "You are about to delete a RAID device."
msgstr "RAID デバイスを削除しようとしています。"

#: partIntfHelpers.py:331
#, python-format
msgid "You are about to delete the %s partition."
msgstr "%s パーティションを削除しようとしています。"

#: partIntfHelpers.py:335
#, fuzzy, python-format
msgid "You are about to delete the %s %s"
msgstr "%(type)s %(name)s を削除しようとしています。"

#: partIntfHelpers.py:346
msgid "Confirm Reset"
msgstr "リセットの確認"

#: partIntfHelpers.py:347
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr "本当に、パーティションテーブル設定をリセットして元の状態に戻しますか?"

#: platform.py:93 platform.py:320 platform.py:388 platform.py:488
msgid "RAID Device"
msgstr "RAID デバイス"

#: platform.py:94 platform.py:97 platform.py:321 platform.py:389
#: platform.py:489 platform.py:492
msgid "Master Boot Record (MBR)"
msgstr "マスタブートレコード(MBR)"

#: platform.py:96 platform.py:491
msgid "First sector of boot partition"
msgstr "ブートパーティションの最初のセクタ"

#: platform.py:112
msgid "You have not created a bootable partition."
msgstr "ブート可能なパーティションが作成されていません。"

#: platform.py:115
msgid "Bootable partitions can only be on RAID1 devices."
msgstr "RAID1 デバイス上にしか起動可能なパーティションは存在できません。"

#: platform.py:119
msgid "Bootable partitions cannot be on a logical volume."
msgstr "起動可能なパーティションは論理ボリューム上には存在できません。"

#: platform.py:123
msgid "Bootable partitions cannot be on a RAID device."
msgstr "起動可能なパーティションを RAID デバイス上に作成することはできません。"

#: platform.py:127 platform.py:131
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr ""
"起動可能なパーティションを、ファイルシステム %s に作成することはできません。"

#: platform.py:135 platform.py:140
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr ""
"起動可能なパーティションを暗号化されたブロックデバイス上に作成することはでき"
"ません。"

#: platform.py:214
msgid "EFI System Partition"
msgstr "EFI システムパーティション"

#: platform.py:219
msgid "You have not created a /boot/efi partition."
msgstr "/boot/efi パーティションが作成されていません。"

#: platform.py:225
#, fuzzy
msgid "/boot is not on an ext2 filesystem."
msgstr ""
"起動可能なパーティションを、ファイルシステム %s に作成することはできません。"

#: platform.py:228
msgid "/boot/efi is not EFI."
msgstr "/boot/efi は EFI ではありません。"

#: platform.py:265
#, python-format
msgid "%s must have a bsd disk label."
msgstr "%s は bsd ディスクラベルを持っている必要があります。"

#: platform.py:277
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr "ディスク %s には最初に少なくとも 1MB の空き領域が必要です。"

#: platform.py:323
msgid "PPC PReP Boot"
msgstr "PPC PReP ブート"

#: platform.py:339
msgid "The boot partition must be within the first 4MB of the disk."
msgstr "ブートパーティションはディスクの最初の 4MB 以内にある必要があります。"

#: platform.py:391 platform.py:394
msgid "Apple Bootstrap"
msgstr "Apple ブートストラップ"

#: platform.py:408
#, python-format
msgid "%s must have a mac disk label."
msgstr "%s には mac ディスクラベルを持っている必要があります。"

#: rescue.py:199
msgid "When finished please exit from the shell and your system will reboot."
msgstr "完了したらシェルを終了してください。システムが再起動します。"

#: rescue.py:213
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr "実行するための /bin/sh が見付かりません。シェルを 開始していません。"

#: rescue.py:232
msgid "Setup Networking"
msgstr "ネットワークの設定"

#: rescue.py:233
msgid "Do you want to start the network interfaces on this system?"
msgstr "このシステムでネットワークインターフェースを開始しますか?"

#: rescue.py:234 loader/driverdisk.c:483 loader/driverdisk.c:493
#: loader/hdinstall.c:222 textw/constants_text.py:56
msgid "Yes"
msgstr "はい"

#: rescue.py:234 rescue.py:236 loader/driverdisk.c:483 loader/driverdisk.c:493
#: textw/constants_text.py:60
msgid "No"
msgstr "いいえ"

#: rescue.py:241
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""
"ネットワーキングデバイスを始動できません。ネットワーキングはレスキューモード"
"では 使用できません。"

#: rescue.py:276 rescue.py:345 rescue.py:356 rescue.py:434
msgid "Rescue"
msgstr "レスキュー"

#: rescue.py:277
#, fuzzy, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"レスキュー環境はこれから Linux インストールを探し、ディレクトリー %s の下にマ"
"ウントしようと試みます。その後にご使用のシステムに必要な変更を行うことができ"
"ます。このプロセスを続ける場合、「続行」を選択してください。また「読み取り専"
"用」を選択すると、「読み取りと書き込み」ではなく「読み取り専用」でのファイル"
"システムのマウントを選択することもできます。SAN デバイスをアクティベート する"
"必要がある場合は、「高度な設定」を選択して下さい。\n"
"\n"
"何らかの原因でこのプロセスが失敗する場合は「スキップする」を選択し、このプロ"
"セスをとばして直接コマンドシェルに行くことができます。\n"
"\n"

#: rescue.py:287 iw/partition_gui.py:544 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/driverdisk.c:444 storage/__init__.py:141
#: storage/devicetree.py:85
msgid "Continue"
msgstr "続行"

#: rescue.py:287 rescue.py:292
msgid "Read-Only"
msgstr "読み取り専用"

#: rescue.py:287 rescue.py:289 loader/cdinstall.c:248 loader/cdinstall.c:251
#: loader/method.c:326 storage/__init__.py:1644 textw/upgrade_text.py:133
msgid "Skip"
msgstr "スキップする"

#: rescue.py:321
msgid "System to Rescue"
msgstr "レスキュー対象システム"

#: rescue.py:322
msgid "Which device holds the root partition of your installation?"
msgstr "どのデバイスにインストールの root パーティションを置きますか？"

#: rescue.py:324 rescue.py:328 text.py:470 text.py:472
msgid "Exit"
msgstr "終了"

#: rescue.py:346
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""
"システムに損傷のあるファイルシステムがありマウントしないことを選択していま"
"す。[Enter] を押すとシェルが起動され、そこで fsck やパーティションのマウント"
"ができます。シェルを終了するとシステムは自動的に再起動します。"

#: rescue.py:357
#, fuzzy, python-format
msgid ""
"Your system has been mounted under %s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""
"システムを %(rootPath)s 下にマウントしています。\n"
"\n"
"シェルを起動するには <return> を押してください。そのシステムを root 環境とし"
"て利用するには次のコマンドを実行します:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"シェルを終了するとシステムは自動的に再起動します。"

#: rescue.py:435
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""
"システムのいくつか、あるいはすべてをマウントしようと試みている間にエラーが発"
"生しました。いくつかのパーティションは %s の下にマウントされています。\n"
"\n"
"シェルを起動するには [Enter] を押してください。シェルを終了するとシステムは自"
"動的に再起動します。"

#: rescue.py:444
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Linux パーティションがありません。再起動します。\n"

#: rescue.py:447
msgid "Rescue Mode"
msgstr "レスキューモード"

#: rescue.py:448
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""
"Linux パーティションがありません。シェルを起動するには [Enter] を押してくださ"
"い。シェルを終了するとシステムは自動的に再起動します。"

#: rescue.py:461
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "システムは %s ディレクトリー配下にマウントされます。"

#: text.py:153
msgid "Passphrase for encrypted device"
msgstr "暗号化されたデバイス用のパスフレーズ"

#: text.py:166
msgid "Also add this passphrase to all existing encrypted devices"
msgstr "全ての暗号化された既存デバイスにもこのパスフレーズを追加します。"

#: text.py:231 ui/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr "パスフレーズ"

#: text.py:239 ui/lukspassphrase.glade.h:7
msgid "This is a global passphrase"
msgstr "これはグローバルパスフレーズです。"

#: text.py:309 text.py:313
msgid "Repository editing is not available in text mode."
msgstr "リポジトリーの編集は、テキストモードではできません。"

#: text.py:375 loader/lang.c:63 loader/loader.c:215
#, c-format, python-format
msgid "Welcome to %s for %s"
msgstr "%s (%s 用)へようこそ"

#: text.py:377
#, python-format
msgid "Welcome to %s"
msgstr "%s へようこそ"

#: text.py:379
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr "  <Tab>/<Alt-Tab> 項目間の移動   |  <Space> 選択   |  <F12> 次の画面"

#: text.py:470 loader/net.c:111 loader/net.c:472 loader/net.c:528
#: loader/net.c:671 loader/net.c:679 loader/net.c:1102 loader/net.c:1109
msgid "Retry"
msgstr "再試行"

#: text.py:496
msgid "Cancelled"
msgstr "取り消されました"

#: text.py:497
msgid "I can't go to the previous step from here. You will have to try again."
msgstr "直前のステップに戻ることはできません。もう一度やり直してください。"

#: upgrade.py:51
msgid "Proceed with upgrade?"
msgstr "アップグレードしますか?"

#: upgrade.py:52
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""
"アップグレードの選択をした Linux インストールのファイルシステムは既にマウント"
"されています。ここから元に戻ることはできません。\n"
"\n"

#: upgrade.py:56
msgid "Would you like to continue with the upgrade?"
msgstr "アップグレードを続行しますか?"

#: upgrade.py:184
msgid "Mount failed"
msgstr "マウントに失敗しました"

#: upgrade.py:185
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""
"/etc/fstab に一覧表示されたファイルシステムをマウント中に以下のエラーが 発生"
"しました。この問題を解決してから再度アップグレードをして下さい\n"
"%s"

#: upgrade.py:193 upgrade.py:200
msgid "Upgrade root not found"
msgstr "アップグレード root が見付かりません"

#: upgrade.py:194
msgid "The root for the previously installed system was not found."
msgstr "以前のインストールされたシステムに root が見つかりませんでした。"

#: upgrade.py:201
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""
"以前にインストールされたシステムの root が見付かりません。インストーラを 終了"
"するか、又はアップグレードの代わりにインストールを選択するために戻る ことが出"
"来ます。"

#: upgrade.py:224
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""
"次のファイルはアップグレード中にはサポートしていない絶対シンボリックリンクで"
"す。相対シンボリックリンクに変更してからアップグレードを再開してください。\n"
"\n"

#: upgrade.py:230
msgid "Absolute Symlinks"
msgstr "絶対 Symlink"

#: upgrade.py:241
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""
"これらは、シンボリックリンクであるべきディレクトリーであり、アップグレード時"
"に起きる問題の原因となります。シンボリックリンクとして本来の状態に戻してから"
"アップグレードを再開してください。\n"
"\n"

#: upgrade.py:247
msgid "Invalid Directories"
msgstr "無効なディレクトリー"

#: upgrade.py:252
#, fuzzy, python-format
msgid "%s not found"
msgstr "ディスクが見つかりません"

#: vnc.py:137
#, fuzzy, python-format
msgid "%s %s installation on host %s"
msgstr "ホスト %(name)s 上の %(productName)s %(productVersion)s インストール"

#: vnc.py:139
#, fuzzy, python-format
msgid "%s %s installation"
msgstr "インストール後"

#: vnc.py:166
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "ホスト %s 上の vnc クライアントに接続を試行中..."

#: vnc.py:180
msgid "Connected!"
msgstr "接続しました!"

#: vnc.py:183
msgid "Will try to connect again in 15 seconds..."
msgstr "15 秒後に再接続を試みます..."

#: vnc.py:200
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"インストールを開始するには、ご使用の vnc クライアントを %s に手動で接続してく"
"ださい。"

#: vnc.py:202
msgid "Please manually connect your vnc client to begin the install."
msgstr ""
"インストールを開始するには、ご使用の vnc クライアントを手動で接続してくださ"
"い。"

#: vnc.py:205
msgid "Starting VNC..."
msgstr "VNC を起動中..."

#: vnc.py:230
msgid "The VNC server is now running."
msgstr "VNC サーバーが稼動しています。"

#: vnc.py:243
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Listen モードの VNC Viewer に接続するよう選択しました。\n"
"これは、パスワードを必要としません。もしパスワードを設定すると、\n"
"VNC Viewew への接続失敗の原因になります。\n"
"\n"

#: vnc.py:248
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"警告!!! VNC サーバーがパスワードなしで実行されています!\n"
"サーバーの安全性を確保するには、vncpassword=<password>\n"
"ブートオプションを使うことができます。\n"
"\n"

#: vnc.py:252
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"パスワードを使って VNC を実行するように選択しました。\n"
"\n"

#: vnc.py:254
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"不明なエラーが発生しました。作業を中断します。\n"

#: vnc.py:276 vnc.py:369
msgid "VNC Configuration"
msgstr "VNC の設定"

#: vnc.py:279 vnc.py:373
msgid "No password"
msgstr "パスワードなし"

#: vnc.py:281 vnc.py:376
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""
"パスワードによって許可のないリスナーが接続してインストールの進捗を傍受するの"
"を防ぎます。インストールに使用するパスワードを入力してください。"

#: vnc.py:289 vnc.py:384 textw/userauth_text.py:47
msgid "Password:"
msgstr "パスワード:"

#: vnc.py:290 vnc.py:385 textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr "パスワード (確認):"

#: vnc.py:308 vnc.py:407 textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr "パスワードが一致しません"

#: vnc.py:309 vnc.py:408 textw/userauth_text.py:71
msgid "The passwords you entered were different. Please try again."
msgstr "入力したパスワードが異なっています。再度やり直してください。"

#: vnc.py:314 vnc.py:413 textw/userauth_text.py:66
msgid "Password Length"
msgstr "パスワードの長さ"

#: vnc.py:315 vnc.py:414
msgid "The password must be at least six characters long."
msgstr "パスワードは最低 6 文字以上でなければなりません。"

#: vnc.py:337
msgid "Unable to Start X"
msgstr "X を起動できません"

#: vnc.py:339
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""
"ご使用のマシンで X を起動できませんでした。VNC を起動して別のコンピューターか"
"らこのコンピューターに接続しグラフィカルインストールを行ないますか、あるいは"
"テキストモードのインストールで続行しますか?"

#: vnc.py:358
msgid "Start VNC"
msgstr "VNC を起動する"

#: vnc.py:359 vnc.py:361
msgid "Use text mode"
msgstr "テキストモードを使用する"

#: yuminstall.py:82
#, python-format
msgid "%s MB"
msgstr "%s MB"

#: yuminstall.py:85
#, python-format
msgid "%s KB"
msgstr "%s KB"

#: yuminstall.py:131
msgid "Preparing to install"
msgstr "インストールの準備中"

#: yuminstall.py:132
msgid "Preparing transaction from installation source"
msgstr "インストールソースからのトランザクションを準備中。"

#: yuminstall.py:160
#, fuzzy, python-format
msgid "<b>Installing %s</b> (%s)\n"
msgstr "<b>%(pkgStr)s のインストール</b> (%(size)s)\n"

#: yuminstall.py:220
msgid "Finishing upgrade"
msgstr "アップグレードを終了中..."

#: yuminstall.py:221
msgid "Finishing upgrade process.  This may take a little while."
msgstr "アップグレードプロセスを終了しています。数分かかることがあります。"

#: yuminstall.py:230
msgid "Error Installing Package"
msgstr "パッケージのインストールでエラー"

#: yuminstall.py:231
#, python-format
msgid ""
"A fatal error occurred when installing the %s package.  This could indicate "
"errors when reading the installation media.  Installation cannot continue."
msgstr ""
"%s パッケージをインストールしている時に致命的なエラーが発生しました。これはイ"
"ンストール媒体を読み込む時のエラーを示しているのかもしれません。インストール"
"を継続できません。"

#: yuminstall.py:332 iw/task_gui.py:317
msgid "Error Setting Up Repository"
msgstr "リポジトリーの設定中にエラー"

#: yuminstall.py:333
#, fuzzy, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%s\n"
"\n"
"Please provide the correct information for installing %s."
msgstr ""
"インストールリポジトリー設定中に次のエラーが発生しました:\n"
"\n"
"%(e)s\n"
"\n"
"%(productName)s のインストールについての正しい情報を提示して下さい。"

#: yuminstall.py:376
msgid "Change Disc"
msgstr "ディスクの変更"

#: yuminstall.py:377
#, fuzzy, python-format
msgid "Please insert %s disc %d to continue."
msgstr "%(productName)s ディスク %(discnum)d を挿入して続行して下さい。"

#: yuminstall.py:387
msgid "Wrong Disc"
msgstr "不適切なディスクです"

#: yuminstall.py:388
#, python-format
msgid "That's not the correct %s disc."
msgstr "正しい %s ディスクではありません。"

#: yuminstall.py:395
msgid "Unable to access the disc."
msgstr "ディスクにアクセスできません。"

#: yuminstall.py:547
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr "リポジトリー %r は設定内で名前がありません。id を使用します。"

#: yuminstall.py:661 yuminstall.py:1078
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""
"ソフトウェアリポジトリーの一部はネットワークを必要とします。しかし、使用中の"
"システムでネットワークの有効化にエラーが発生しました。"

#: yuminstall.py:724 yuminstall.py:726
msgid "Re_boot"
msgstr "再起動(_B)"

#: yuminstall.py:724
msgid "_Eject"
msgstr "ディスクを取り出す (_E)"

#: yuminstall.py:735
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""
"ファイル %s を開けません。ファイルが見つからないか、パッケージやメディアが破"
"損している可能性があります。インストール元を検証してください。\n"
"\n"
"終了するとシステムは矛盾した状態となり、再インストールを必要とする可能性が高"
"くなります。\n"
"\n"

#: yuminstall.py:783
msgid "Retrying"
msgstr "再試行中"

#: yuminstall.py:783
msgid "Retrying download."
msgstr "ダウンロードの再試行中。"

#: yuminstall.py:849
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr "トランザクションの実行中に次のような理由でエラーが発生しました。%s\n"

#: yuminstall.py:884 yuminstall.py:885
msgid "file conflicts"
msgstr "ファイル競合"

#: yuminstall.py:886
msgid "older package(s)"
msgstr "旧パッケージ"

#: yuminstall.py:887
msgid "insufficient disk space"
msgstr "ディスク容量の不足"

#: yuminstall.py:888
msgid "insufficient disk inodes"
msgstr "ディスク inode の不足"

#: yuminstall.py:889
msgid "package conflicts"
msgstr "パッケージ競合"

#: yuminstall.py:890
msgid "package already installed"
msgstr "パッケージは既にインストールされています"

#: yuminstall.py:891
msgid "required package"
msgstr "必要なパッケージ"

#: yuminstall.py:892
msgid "package for incorrect arch"
msgstr "不適切なアーキテクチャ用のパッケージ"

#: yuminstall.py:893
msgid "package for incorrect os"
msgstr "不適切な OS 用のパッケージ"

#: yuminstall.py:907
msgid "You need more space on the following file systems:\n"
msgstr "以下のファイルシステムは容量を増やす必要があります。\n"

#: yuminstall.py:920
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""
"インストールするパッケージに、競合するファイルがみつかりました:\n"
"%s\n"

#: yuminstall.py:923
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): %"
"s.\n"
msgstr "トランザクションの実行中に次のような理由でエラーが発生しました: %s\n"

#: yuminstall.py:930 yuminstall.py:935
msgid "Error Running Transaction"
msgstr "トランザクション実行中にエラーが発生しました。"

#: yuminstall.py:1112
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""
"リポジトリーからグループ情報を読み取ることはできません。これは、インストール"
"ツリーの生成の問題です。"

#: yuminstall.py:1148
msgid "Retrieving installation information."
msgstr "インストール情報を読み出し中。"

#: yuminstall.py:1150
#, python-format
msgid "Retrieving installation information for %s."
msgstr "%s に関するインストール情報を読み出し中。"

#: yuminstall.py:1152
msgid "Installation Progress"
msgstr "インストールの進捗"

#: yuminstall.py:1165 textw/constants_text.py:64
msgid "Edit"
msgstr "編集"

#: yuminstall.py:1176 yuminstall.py:1335
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""
"パッケージのメタデータを読めません。repodata ディレクトリーが見つからないのが"
"原因かもしれません。インストールツリーが正しく構築されているか確認してくださ"
"い。\n"
"\n"
"%s"

#: yuminstall.py:1353
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""
"選択したパッケージはインストールに %d MB の空き領域を必要としますが、充分な空"
"きがありません。選択を変更するか、またはインストーラーを終了してください。"

#: yuminstall.py:1372
msgid "Reboot?"
msgstr "再起動しますか?"

#: yuminstall.py:1373
msgid "The system will be rebooted now."
msgstr "システムが再起動します。"

#: yuminstall.py:1516
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"%s のこのバージョンにアップグレードを行うには古過ぎるシステムからアップグレー"
"トしているようです。本当にこのアップグレードのプロセスを続行しますか?"

#: yuminstall.py:1552
#, fuzzy, python-format
msgid ""
"The arch of the release of %s you are upgrading to appears to be %s which "
"does not match your previously installed arch of %s.  This is likely to not "
"succeed.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"アップグレードしようとしている %(productName)s のリリースパッケージのアーキテ"
"クチャは %(myarch)s で、既にインストールされているパッケージのアーキテク"
"チャ、%(arch)s と一致しません。この状況では、インストールに失敗するかもしれま"
"せん。本当にこのアップグレードプロセスを続行しますか?"

#: yuminstall.py:1603
msgid "Post Upgrade"
msgstr "アップグレード後"

#: yuminstall.py:1604
msgid "Performing post-upgrade configuration"
msgstr "アップグレード後の設定を実行中。"

#: yuminstall.py:1606
msgid "Post Installation"
msgstr "インストール後"

#: yuminstall.py:1607
msgid "Performing post-installation configuration"
msgstr "インストール後の設定を実行中"

#: yuminstall.py:1828
msgid "Installation Starting"
msgstr "インストールを開始しています"

#: yuminstall.py:1829
msgid "Starting installation process"
msgstr "インストール処理の開始"

#: yuminstall.py:1867
msgid "Dependency Check"
msgstr "依存関係チェック"

#: yuminstall.py:1868
msgid "Checking dependencies in packages selected for installation"
msgstr "インストール対象パッケージの依存関係をチェック中。"

#: installclasses/fedora.py:39
msgid "_Fedora"
msgstr "Fedora(_F)"

#: installclasses/fedora.py:40
#, fuzzy, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. What additional tasks would you like your system to "
"include support for?"
msgstr ""
"%s のデフォルトインストールには一般的なインターネット使用に適用できるソフト"
"ウェアのセットが含まれています。システムにサポートさせる追加タスクを指定して"
"ください。"

#: installclasses/fedora.py:49
msgid "Office and Productivity"
msgstr ""

#: installclasses/fedora.py:50 installclasses/rhel.py:58
msgid "Software Development"
msgstr "ソフトウェア開発"

#: installclasses/fedora.py:51 installclasses/rhel.py:68
msgid "Web Server"
msgstr "Web サーバー"

#: installclasses/rhel.py:40
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: installclasses/rhel.py:41
#, fuzzy, python-format
msgid ""
"The default installation of %s is a minimal install. What additional tasks "
"would you like your system to include support for?"
msgstr ""
"%s のデフォルトインストールには一般的なインターネット使用に適用できるソフト"
"ウェアのセットが含まれています。システムにサポートさせる追加タスクを指定して"
"ください。"

#: installclasses/rhel.py:49
msgid "Desktop"
msgstr ""

#: installclasses/rhel.py:75
#, fuzzy
msgid "Advanced Server"
msgstr "高度な設定"

#: iw/GroupSelector.py:147
#, python-format
msgid "Packages in %s"
msgstr "%s に含まれるパッケージ"

#: iw/GroupSelector.py:424
#, fuzzy, python-format
msgid "Optional packages selected: %d of %d"
msgstr "オプションパッケージが選択されました: %(cnt)d 個内の %(inst)d 個 "

#: iw/GroupSelector.py:425
#, python-format
msgid "<i>%s</i>"
msgstr "<i>%s</i>"

#: iw/GroupSelector.py:485
msgid "Uncategorized"
msgstr "カテゴリなし"

#: iw/account_gui.py:52
msgid "Root _Password:"
msgstr "ルートパスワード (_P):"

#: iw/account_gui.py:54
msgid "_Confirm:"
msgstr "確認 (_C):"

#: iw/account_gui.py:92
msgid "Caps Lock is on."
msgstr "Caps Lock がオンになっています。"

#: iw/account_gui.py:102 iw/account_gui.py:110 iw/account_gui.py:117
#: iw/account_gui.py:137 textw/userauth_text.py:74
msgid "Error with Password"
msgstr "パスワードにエラー"

#: iw/account_gui.py:103
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""
"続行するには、 root パスワードを入力してから、もう一度同じパスワードを入力し"
"て確認しなければなりません。"

#: iw/account_gui.py:111
msgid "The passwords you entered were different.  Please try again."
msgstr "入力したパスワードが異ります。やり直してください。"

#: iw/account_gui.py:118
msgid "The root password must be at least six characters long."
msgstr "root パスワードは最低 6 文字以上でなければなりません。"

#: iw/account_gui.py:127 textw/userauth_text.py:83
msgid "Weak Password"
msgstr "パスワードが弱すぎます。"

#: iw/account_gui.py:128
#, python-format
msgid "You have provided a weak password: %s"
msgstr "あなたは弱いパスワードを与えています: %s"

#: iw/account_gui.py:130
msgid "Use Anyway"
msgstr "とにかく使う"

#: iw/account_gui.py:138 textw/userauth_text.py:75
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""
"要求されたパスワードには、パスワードに使用できない非 ASCII 文字が含まれていま"
"す。"

#: iw/autopart_type.py:100
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""
"リサイズできるパーティションがありません。特定のファイルシステムでフォーマッ"
"トされた、物理パーティションのみリサイズできます。"

#: iw/autopart_type.py:123
msgid "Resize FileSystem Error"
msgstr "ファイルシステムリサイズエラー"

#: iw/autopart_type.py:124 iw/autopart_type.py:133
#, python-format
msgid "%s: %s"
msgstr ""

#: iw/autopart_type.py:132
msgid "Resize Device Error"
msgstr "デバイスリサイズエラー"

#: iw/autopart_type.py:207
msgid ""
"Do you really want to boot from a disk which is not used for installation?"
msgstr ""

#: iw/autopart_type.py:284
msgid "Invalid Initiator Name"
msgstr "無効なイニシエーター名です"

#: iw/autopart_type.py:285
msgid "You must provide an initiator name."
msgstr "イニシエーター名を入力しなければなりません。"

#: iw/autopart_type.py:318
msgid "Error with Data"
msgstr "データにエラー発生"

#: iw/autopart_type.py:476 iw/autopart_type.py:477
#, fuzzy
msgid "Rescanning disks"
msgstr "ドライバーディスクを読み込み中"

#: iw/autopart_type.py:531 textw/partition_text.py:63
msgid "Use entire drive"
msgstr "ドライブ全域を使用"

#: iw/autopart_type.py:532 textw/partition_text.py:64
msgid "Replace existing Linux system"
msgstr "既存の Linux システムを入れ替え"

#: iw/autopart_type.py:533
#, fuzzy
msgid "Shrink current system"
msgstr "現在のシステムを縮小する"

#: iw/autopart_type.py:534 textw/partition_text.py:65
msgid "Use free space"
msgstr "空き領域を使用"

#: iw/autopart_type.py:535
#, fuzzy
msgid "Create custom layout"
msgstr "カスタムレイアウトを作成する"

#: iw/blpasswidget.py:44
msgid "_Use a boot loader password"
msgstr "ブートローダーパスワードを使用(_U)"

#: iw/blpasswidget.py:45
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""
"ブートローダーパスワードによって、ユーザがカーネルオプションを変更してしまう"
"のを防ぎ、より高度なシステムセキュリティを確保します。"

#: iw/blpasswidget.py:76
msgid "Change _password"
msgstr "パスワードを変更(_P)"

#: iw/blpasswidget.py:99
msgid "Enter Boot Loader Password"
msgstr "ブートローダーパスワードを入力"

#: iw/blpasswidget.py:105
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""
"ブートローダーのパスワードを入力してからそのパスワードの確認をしてください。"
"(BIOS のキーマップが実際に使用しているキーマップとは異なっている可能性がある"
"ので注意してください)"

#: iw/blpasswidget.py:112
msgid "_Password:"
msgstr "パスワード(_P):"

#: iw/blpasswidget.py:118
msgid "Con_firm:"
msgstr "確認(_F):"

#: iw/blpasswidget.py:139
msgid "Passwords don't match"
msgstr "パスワードが一致しません"

#: iw/blpasswidget.py:140
msgid "Passwords do not match"
msgstr "パスワードが一致しません"

#: iw/blpasswidget.py:149
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"ブートローダーパスワードが 6 文字未満になっています。ブートローダーのパスワー"
"ドはこれより長くすることを推奨します。\n"
"\n"
"このパスワードで続行しますか?"

#: iw/bootloader_main_gui.py:36
msgid "Boot Loader Configuration"
msgstr "ブートローダーの設定"

#: iw/bootloader_main_gui.py:179 iw/bootloader_main_gui.py:184
#: iw/bootloader_main_gui.py:221
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr "ブートローダーを /dev/%s にインストールする。 (_I)"

#: iw/bootloader_main_gui.py:227
msgid "_Change device"
msgstr "デバイスの変更 (_C)"

#: iw/congrats_gui.py:33
msgid "Congratulations"
msgstr "おめでとうございます"

#: iw/congrats_gui.py:74 textw/complete_text.py:39
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""
"おめでとうございます。 %s のインストールは完了しました。\n"
"\n"

#: iw/congrats_gui.py:77 textw/complete_text.py:42
msgid "Shutdown"
msgstr "シャットダウン"

#: iw/congrats_gui.py:79 textw/complete_text.py:44
msgid "Please shutdown to use the installed system.\n"
msgstr "インストールしたシステムを使用するにはシャットダウンして下さい \n"

#: iw/congrats_gui.py:81 textw/complete_text.py:46
msgid "Please reboot to use the installed system.\n"
msgstr "インストール済のシステムを使用するには再起動して下さい。\n"

#: iw/congrats_gui.py:86 textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""
"ご使用のシステムの正常な機能を確実にするために更新が利用できる可能性があり、 "
"再起動の後にこの更新が推奨されることに注意して下さい。"

#: iw/congrats_gui.py:90 textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""
"おめでとうございます。%s のインストールが完了しました。\n"
"\n"
"インストールしたシステムを使用するために再起動して下さい。ご使用の システムの"
"正常な機能を確実にするために更新が利用できる可能性が あり、再起動の後にこの更"
"新が推奨されます。"

#: iw/examine_gui.py:37
msgid "Upgrade Examine"
msgstr "アップグレードテスト"

#: iw/examine_gui.py:59
#, fuzzy, python-format
msgid "_Install %s"
msgstr "%s のインストール\n"

#: iw/examine_gui.py:61
#, fuzzy
msgid ""
"Choose this option to freshly install your system. Existing software and "
"data may be overwritten depending on your configuration choices."
msgstr ""
"システムに %s のコピーを新規インストールするにはこのオプションを選択します。"
"既存のソフトウェアやデータは選択した設定によっては上書きされるかもしれませ"
"ん。"

#: iw/examine_gui.py:65
#, fuzzy
msgid "_Upgrade an existing installation"
msgstr "既存インストールのアップグレード"

#: iw/examine_gui.py:67
#, fuzzy, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option preserves the existing data on your drives."
msgstr ""
"既存の %s システムをアップグレードする場合に、このオプションを選択します。 こ"
"のオプションはストレージデバイス上の既存データを保持します。"

#: iw/examine_gui.py:123
msgid "The following installed system will be upgraded:"
msgstr ""

#: iw/examine_gui.py:136
msgid "Unknown Linux system"
msgstr "不明な Linux システム"

#: iw/language_gui.py:33
msgid "Language Selection"
msgstr "言語の選択"

#: iw/language_gui.py:81 loader/lang.c:377
msgid "What language would you like to use during the installation process?"
msgstr "インストール時に使用する言語を選択してください。"

#: iw/lvm_dialog_gui.py:124 iw/lvm_dialog_gui.py:170 iw/lvm_dialog_gui.py:182
#: iw/lvm_dialog_gui.py:221 iw/lvm_dialog_gui.py:298 iw/lvm_dialog_gui.py:681
#: iw/lvm_dialog_gui.py:701
msgid "Not enough space"
msgstr "容量が足りません"

#: iw/lvm_dialog_gui.py:125
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""
"物理エクステントサイズは変更できません。変更すると現在定義されている論理ボ"
"リュームが必要とする領域が使用可能な領域以上に増加してしまうためです。"

#: iw/lvm_dialog_gui.py:134
msgid "Confirm Physical Extent Change"
msgstr "物理エクステントの変更を確認"

#: iw/lvm_dialog_gui.py:135
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""
"この物理エクステントの値の変更により、現在の論理ボリューム要求サイズを物理エ"
"クステントの整数倍に切り上げる必要があります。\n"
"\n"
"この変更は直ちに反映されます。"

#: iw/lvm_dialog_gui.py:144 iw/lvm_dialog_gui.py:204
msgid "C_ontinue"
msgstr "続行 (_O)"

#: iw/lvm_dialog_gui.py:171
#, fuzzy, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is larger than the smallest physical volume (%10.2f MB) in the "
"volume group."
msgstr ""
"物理エクステントのサイズは変更できません。選択された値 (%(curpe)10.2f MB) が"
"ボリュームグループ内の最小物理ボリューム (%(maxpvsize)10.2f MB) よりも大きい"
"ためです。"

#: iw/lvm_dialog_gui.py:183
#, fuzzy, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is too large compared to the size of the smallest physical volume "
"(%10.2f MB) in the volume group."
msgstr ""
"物理エクステントのサイズを変更することはできません。ボリュームグループ内の最"
"小物理ボリューム (%(maxpvsize)10.2f MB) のサイズと比較して、選択された値 (%"
"(curpe)10.2f MB) が大きすぎるためです。"

#: iw/lvm_dialog_gui.py:197
msgid "Too small"
msgstr "小さすぎます"

#: iw/lvm_dialog_gui.py:198
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""
"物理エクステントの値を変更すると、ボリュームグループ内の1つまたは複数の物理ボ"
"リュームの相当な領域を浪費することになります。"

#: iw/lvm_dialog_gui.py:222
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""
"物理エクステントのサイズは変更できません。結果としてできる最大論理ボリューム"
"サイズ (%10.2f MB) が現在定義されている 1 つまたは複数の論理ボリュームよりも"
"小さくなるためです。"

#: iw/lvm_dialog_gui.py:299
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""
"この物理ボリュームは削除できません。削除すると現在定義されている論理ボリュー"
"ムを保持するにはボリュームグループが小さくなりすぎるためです。"

#: iw/lvm_dialog_gui.py:398
msgid "Make Logical Volume"
msgstr "論理ボリュームを作成"

#: iw/lvm_dialog_gui.py:400
#, python-format
msgid "Edit Logical Volume: %s"
msgstr "論理ボリュームを編集: %s"

#: iw/lvm_dialog_gui.py:431 iw/raid_dialog_gui.py:397
msgid "_File System Type:"
msgstr "ファイルシステムタイプ(_F):"

#: iw/lvm_dialog_gui.py:437
msgid "_Logical Volume Name:"
msgstr "論理ボリューム名(_L):"

#: iw/lvm_dialog_gui.py:446 iw/partition_dialog_gui.py:434
msgid "_Size (MB):"
msgstr "サイズ (MB)(_S):"

#: iw/lvm_dialog_gui.py:453
#, python-format
msgid "(Max size is %s MB)"
msgstr "(最大サイズは %s MB)"

#: iw/lvm_dialog_gui.py:458 iw/partition_dialog_gui.py:486
#: iw/partition_ui_helpers_gui.py:329 iw/raid_dialog_gui.py:369
msgid "_Encrypt"
msgstr "暗号化 (_E)"

#: iw/lvm_dialog_gui.py:467 iw/raid_dialog_gui.py:407
msgid "Original File System Type:"
msgstr "元のファイルシステムタイプ:"

#: iw/lvm_dialog_gui.py:471 iw/partition_gui.py:777 iw/partition_gui.py:929
#: iw/raid_dialog_gui.py:412
#, fuzzy
msgid "Unknown"
msgstr "不明なデバイス"

#: iw/lvm_dialog_gui.py:475 iw/partition_dialog_gui.py:424
#: iw/raid_dialog_gui.py:418
msgid "Original File System Label:"
msgstr "元のファイルシステムラベル:"

#: iw/lvm_dialog_gui.py:480
msgid "Logical Volume Name:"
msgstr "論理ボリューム名:"

#: iw/lvm_dialog_gui.py:484
msgid "Size (MB):"
msgstr "サイズ (MB):"

#: iw/lvm_dialog_gui.py:511 iw/partition_dialog_gui.py:379
#: iw/raid_dialog_gui.py:388
msgid "_Mount Point:"
msgstr "マウントポイント(_M):"

#: iw/lvm_dialog_gui.py:594
msgid "Illegal Logical Volume Name"
msgstr "不適切な論理ボリューム名"

#: iw/lvm_dialog_gui.py:609
msgid "Illegal logical volume name"
msgstr "不適切な論理ボリューム名"

#: iw/lvm_dialog_gui.py:610
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""
"論理ボリューム名 \"%s\" はすでに使用中です。他の名前を選択してください。"

#: iw/lvm_dialog_gui.py:652 iw/partition_dialog_gui.py:126
#: iw/raid_dialog_gui.py:180
msgid "Mount point in use"
msgstr "使用中のマウントポイント"

#: iw/lvm_dialog_gui.py:653 iw/partition_dialog_gui.py:127
#: iw/raid_dialog_gui.py:181
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr "マウントポイント \"%s\" は使用中です。他を選択してください。"

#: iw/lvm_dialog_gui.py:668
msgid "Illegal size"
msgstr "不適切なサイズ"

#: iw/lvm_dialog_gui.py:669
msgid "The requested size as entered is not a valid number greater than 0."
msgstr "入力された要求サイズは 0 より大きい有効な数値ではありません。"

#: iw/lvm_dialog_gui.py:682
#, fuzzy, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the maximum logical "
"volume size (%10.2f MB). To increase this limit you can create more Physical "
"Volumes from unpartitioned disk space and add them to this Volume Group."
msgstr ""
"現在の要求サイズ (%(size)10.2f MB) は、最大論理ボリュームサイズ (%(maxlv)"
"10.2f MB) より大きくなります。この上限を増やすには、パーティションが未設定の"
"ディスク領域から物理ボリュームを作成し、それをこのボリュームグループに追加し"
"ます。"

#: iw/lvm_dialog_gui.py:702
#, fuzzy, python-format
msgid ""
"The logical volumes you have configured require %d MB, but the volume group "
"only has %d MB.  Please either make the volume group larger or make the "
"logical volume(s) smaller."
msgstr ""
"設定している論理ボリュームには %(size)d MB 必要ですが、ボリュームグループに"
"は %(tempvgsize)d MB しかありません。ボリュームグループを大きくするか、論理ボ"
"リュームを 小さくしてください。"

#: iw/lvm_dialog_gui.py:792
msgid "No free slots"
msgstr "空きスロットがありません"

#: iw/lvm_dialog_gui.py:803
msgid "No free space"
msgstr "空き領域はありません"

#: iw/lvm_dialog_gui.py:804
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""
"新規の論理ボリュームを作成する空き領域がボリュームグループには残っていませ"
"ん。論理ボリュームを追加するには、ひとつ以上の既存の論理ボリュームのサイズを"
"縮小する必要があります。"

#: iw/lvm_dialog_gui.py:839
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr "本当に論理ボリューム 「%s」 を削除しますか?"

#: iw/lvm_dialog_gui.py:945
msgid "Invalid Volume Group Name"
msgstr "無効なボリュームグループ名"

#: iw/lvm_dialog_gui.py:954
msgid "Name in use"
msgstr "名前は使用中です"

#: iw/lvm_dialog_gui.py:955
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr ""
"ボリュームグループ名 \"%s\" はすでに使用されています。別の名前を選択してくだ"
"さい。"

#: iw/lvm_dialog_gui.py:1201
msgid "Not enough physical volumes"
msgstr "物理ボリュームが足りません"

#: iw/lvm_dialog_gui.py:1202
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""
"LVM ボリュームグループを作成するには、未使用の物理ボリュームパーティションが"
"少なくとも 1 つ必要です。\n"
"\n"
"「物理ボリューム (LVM)」タイプのパーティションまたは RAID アレイを作成してか"
"ら、 「LVM」 オプションを選択し直してください。"

#: iw/lvm_dialog_gui.py:1213
msgid "Make LVM Volume Group"
msgstr "LVM ボリュームグループの作成"

#: iw/lvm_dialog_gui.py:1216
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr "LVM ボリュームグループの編集: %s"

#: iw/lvm_dialog_gui.py:1218
msgid "Edit LVM Volume Group"
msgstr "LVM ボリュームグループの編集"

#: iw/lvm_dialog_gui.py:1234
msgid "_Volume Group Name:"
msgstr "ボリュームグループ名(_V):"

#: iw/lvm_dialog_gui.py:1242
msgid "Volume Group Name:"
msgstr "ボリュームグループ名:"

#: iw/lvm_dialog_gui.py:1250
msgid "_Physical Extent:"
msgstr "物理エクステント(_P): "

#: iw/lvm_dialog_gui.py:1265
msgid "Physical Volumes to _Use:"
msgstr "使用する物理ボリューム(_U):"

#: iw/lvm_dialog_gui.py:1271
msgid "Used Space:"
msgstr "使用中の領域:"

#: iw/lvm_dialog_gui.py:1288
msgid "Free Space:"
msgstr "空き領域:"

#: iw/lvm_dialog_gui.py:1306
msgid "Total Space:"
msgstr "合計領域:"

#: iw/lvm_dialog_gui.py:1344
msgid "Logical Volume Name"
msgstr "論理ボリューム名"

#: iw/lvm_dialog_gui.py:1347 iw/partition_gui.py:363
#: iw/upgrade_swap_gui.py:139 textw/upgrade_text.py:118
msgid "Mount Point"
msgstr "マウントポイント"

#: iw/lvm_dialog_gui.py:1350 iw/partition_gui.py:368
msgid "Size (MB)"
msgstr "サイズ (MB)"

#: iw/lvm_dialog_gui.py:1364 iw/osbootwidget.py:95
msgid "_Add"
msgstr "追加(_A)"

#: iw/lvm_dialog_gui.py:1367 iw/osbootwidget.py:99 iw/partition_gui.py:1347
msgid "_Edit"
msgstr "編集(_E)"

#: iw/lvm_dialog_gui.py:1382
msgid "_Logical Volumes"
msgstr "論理ボリューム (_L)"

#: iw/netconfig_dialog.py:192 textw/netconfig_text.py:36
#, fuzzy, python-format
msgid ""
"An error occurred converting the value entered for \"%s\":\n"
"%s"
msgstr ""
"\"%(field)s\" に入力された値を変換中にエラーが発生:\n"
"%(errmsg)s"

#: iw/netconfig_dialog.py:194 iw/netconfig_dialog.py:203
#: textw/netconfig_text.py:35 textw/netconfig_text.py:40
msgid "Error With Data"
msgstr "データにエラーがあります"

#: iw/netconfig_dialog.py:202
#, python-format
msgid "A value is required for the field %s."
msgstr "フィールド %s には値が必要です。"

#: iw/netconfig_dialog.py:212
#, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr "%s ネットワークインターフェイスを起動中にエラーが発生しました。"

#: iw/netconfig_dialog.py:214
msgid "Error Configuring Network"
msgstr "ネットワーク設定中にエラー発生"

#: iw/netconfig_dialog.py:240
msgid "Dynamic IP Address"
msgstr "動的 IP アドレス"

#: iw/netconfig_dialog.py:241
#, python-format
msgid "Sending request for IP address information for %s"
msgstr "%s の IP アドレス情報の要求を送信中"

#: iw/netconfig_dialog.py:256 iw/netconfig_dialog.py:259
#: textw/netconfig_text.py:223 textw/netconfig_text.py:226
msgid "IP Address"
msgstr "IP アドレス"

#: iw/netconfig_dialog.py:266 textw/netconfig_text.py:233
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr "IPv4 CIDR の接頭辞は 0 から 32 の間にしてください。"

#: iw/netconfig_dialog.py:267 iw/netconfig_dialog.py:273
#: iw/netconfig_dialog.py:281 iw/netconfig_dialog.py:284
#: textw/netconfig_text.py:234 textw/netconfig_text.py:240
#: textw/netconfig_text.py:248
msgid "IPv4 Network Mask"
msgstr "IPv4 ネットワークマスク"

#: iw/netconfig_dialog.py:294 textw/netconfig_text.py:261
msgid "Gateway"
msgstr "ゲートウェイ"

#: iw/netconfig_dialog.py:304 textw/netconfig_text.py:271
msgid "Nameserver"
msgstr "ネームサーバー"

#: iw/netconfig_dialog.py:313
msgid "Error configuring network device:"
msgstr "ネットワークデバイス設定中にエラー発生:"

#: iw/network_gui.py:67 iw/network_gui.py:73
msgid "Error with Hostname"
msgstr "ホスト名にエラー発生"

#: iw/network_gui.py:68
msgid "You must enter a valid hostname for this computer."
msgstr "このコンピュータ用の有効なホスト名を入力して下さい。"

#: iw/network_gui.py:74
#, fuzzy, python-format
msgid ""
"The hostname \"%s\" is not valid for the following reason:\n"
"\n"
"%s"
msgstr ""
"ホスト名 \"%(hostname)s\" は、次のような理由で有効ではありません。\n"
"\n"
"%(herrors)s"

#: iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr "ブートローダーのオペレーティングシステムのリスト"

#: iw/osbootwidget.py:66
msgid "Default"
msgstr "デフォルト"

#: iw/osbootwidget.py:66 iw/partition_gui.py:362
msgid "Label"
msgstr "ラベル"

#: iw/osbootwidget.py:66 iw/partition_gui.py:361
msgid "Device"
msgstr "デバイス"

#: iw/osbootwidget.py:129
msgid "Image"
msgstr "イメージ"

#: iw/osbootwidget.py:136
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""
"ブートローダーメニューに表示されるラベルを入力します。デバイス(または、ハード"
"ドライブとパーティション番号)とは、起動用に使うデバイスのことです。"

#: iw/osbootwidget.py:144
msgid "_Label"
msgstr "ラベル(_L)"

#: iw/osbootwidget.py:152
msgid "_Device"
msgstr "デバイス(_D)"

#: iw/osbootwidget.py:182
msgid "Default Boot _Target"
msgstr "デフォルトのブートターゲット(_T)"

#: iw/osbootwidget.py:211
msgid "You must specify a label for the entry"
msgstr "ブートエントリのラベルを指定してください"

#: iw/osbootwidget.py:220
msgid "Boot label contains illegal characters"
msgstr "ブートラベルに不適切な文字があります"

#: iw/osbootwidget.py:244
msgid "Duplicate Label"
msgstr "ラベルが重複しています"

#: iw/osbootwidget.py:245
msgid "This label is already in use for another boot entry."
msgstr "このラベルは既に別のブートエントリに使用されています。"

#: iw/osbootwidget.py:258
msgid "Duplicate Device"
msgstr "デバイスが重複しています"

#: iw/osbootwidget.py:259
msgid "This device is already being used for another boot entry."
msgstr "このデバイスは既に別のブートエントリに使用されています。"

#: iw/osbootwidget.py:321
msgid "Cannot Delete"
msgstr "削除できません"

#: iw/osbootwidget.py:322
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""
"このブートターゲットは、インストールしようとしている %s システム用になってい"
"るため削除できません。"

#: iw/partition_dialog_gui.py:57
msgid "Additional Size Options"
msgstr "追加容量オプション"

#: iw/partition_dialog_gui.py:62
msgid "_Fixed size"
msgstr "固定容量(_F)"

#: iw/partition_dialog_gui.py:64
msgid "Fill all space _up to (MB):"
msgstr "指定限度まで使用(MB)(_U):"

#: iw/partition_dialog_gui.py:74
msgid "Fill to maximum _allowable size"
msgstr "最大許容量まで使用(_A)"

#: iw/partition_dialog_gui.py:349
msgid "Add Partition"
msgstr "パーティションの追加"

#: iw/partition_dialog_gui.py:351
#, python-format
msgid "Edit Partition: %s"
msgstr "パーティションの編集: %s"

#: iw/partition_dialog_gui.py:388
msgid "File System _Type:"
msgstr "ファイルシステムタイプ(_T):"

#: iw/partition_dialog_gui.py:404
msgid "Allowable _Drives:"
msgstr "使用可能なドライブ(_D):"

#: iw/partition_dialog_gui.py:473
msgid "Force to be a _primary partition"
msgstr "基本パーティションにする(_P)"

#: iw/partition_gui.py:320
#, fuzzy, python-format
msgid "Drive %s (%-0.f MB) (Model: %s)"
msgstr "ドライブ %(drive)s (%(size)-0.f MB) (モデル: %(model)s)"

#: iw/partition_gui.py:364
msgid "Type"
msgstr "タイプ"

#: iw/partition_gui.py:367 storage/__init__.py:1644
msgid "Format"
msgstr "フォーマットする"

#: iw/partition_gui.py:405
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""
"マウントポイント/\n"
"RAID/ボリューム"

#: iw/partition_gui.py:407
msgid ""
"Size\n"
"(MB)"
msgstr ""
"容量\n"
"(MB)"

#: iw/partition_gui.py:519
msgid "Partitioning"
msgstr "パーティション設定"

#: iw/partition_gui.py:609
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""
"あなたが要求したパーティション設定案は以下の重大なエラーを起こしました。"

#: iw/partition_gui.py:611
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr ""
"%s のインストールを続行する前にこれらのエラーを修正する必要があります。"

#: iw/partition_gui.py:617
msgid "Partitioning Errors"
msgstr "パーティション設定エラー"

#: iw/partition_gui.py:624
msgid "The partitioning scheme you requested generated the following warnings."
msgstr "あなたが要求したパーティション設定案には以下の警告があります。"

#: iw/partition_gui.py:626
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr "このパーティション設定のまま続行しますか?"

#: iw/partition_gui.py:631
msgid "Partitioning Warnings"
msgstr "パーティション設定の警告"

#: iw/partition_gui.py:652
msgid "Format Warnings"
msgstr "フォーマットの警告"

#: iw/partition_gui.py:657
msgid "_Format"
msgstr "フォーマット(_F)"

#: iw/partition_gui.py:684
msgid "LVM Volume Groups"
msgstr "LVM ボリュームグループ"

#: iw/partition_gui.py:731
msgid "RAID Devices"
msgstr "RAID デバイス"

#: iw/partition_gui.py:798 loader/hdinstall.c:222
msgid "Hard Drives"
msgstr "ハードドライブ"

#: iw/partition_gui.py:897
msgid "Extended"
msgstr "拡張領域"

#: iw/partition_gui.py:899
#, fuzzy
msgid "software RAID"
msgstr "ソフトウェア RAID を作成"

#: iw/partition_gui.py:931
msgid "Free"
msgstr "空き"

#: iw/partition_gui.py:1024 storage/partitioning.py:190
#: storage/partitioning.py:231
msgid "Error Partitioning"
msgstr "パーティション設定エラー"

#: iw/partition_gui.py:1025
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "要求されたパーティションを設定できませんでした: %s"

#: iw/partition_gui.py:1034
#, python-format
msgid "Warning: %s."
msgstr "警告: %s"

#: iw/partition_gui.py:1064 iw/partition_gui.py:1071
msgid "Unable To Edit"
msgstr "編集できません"

#: iw/partition_gui.py:1065
#, fuzzy
msgid "You must select a device to edit"
msgstr "使用する NIC を選択する必要があります。"

#: iw/partition_gui.py:1072
#, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""
"このデバイスは編集できません:\n"
"\n"
"%s"

#: iw/partition_gui.py:1189 iw/partition_gui.py:1201
msgid "Not supported"
msgstr ""

#: iw/partition_gui.py:1190
msgid "LVM is NOT supported on this platform."
msgstr ""

#: iw/partition_gui.py:1202
msgid "Software RAID is NOT supported on this platform."
msgstr ""

#: iw/partition_gui.py:1209
msgid "No RAID minor device numbers available"
msgstr ""

#: iw/partition_gui.py:1210
msgid ""
"A software RAID device cannot be created because all of the available RAID "
"minor device numbers have been used."
msgstr ""

#: iw/partition_gui.py:1222
#, fuzzy
msgid "RAID Options"
msgstr "オプション(_O)"

#: iw/partition_gui.py:1237
#, fuzzy, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation."
msgstr ""
"ソフトウェア RAID を使用すれば、複数のディスクを結合して 1 つの大きな RAID デ"
"バイスとすることができます。RAID デバイスは、個別のドライブ使用に比べ速度と信"
"頼性を高めるように設定することができます。RAID デバイスの詳細情報については %"
"s ドキュメントを参照してください。\n"

#: iw/partition_gui.py:1247
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""
"RAIDを使用するには、まず「ソフトウェア RAID」タイプのパーティションを少くと"
"も 2 つ作成する必要があります。そうすると、フォーマット及びマウントが可能な "
"RAID デバイスが作成できるようになります。\n"
"\n"

#: iw/partition_gui.py:1253
#, fuzzy
msgid "What do you want to do now?"
msgstr "どうしますか?"

#: iw/partition_gui.py:1262
#, fuzzy
msgid "Create a software RAID _partition."
msgstr "ソフトウェア RAID を作成"

#: iw/partition_gui.py:1265
#, python-format
msgid "Create a RAID _device [default=/dev/md%s]."
msgstr ""

#: iw/partition_gui.py:1269
#, python-format
msgid "Clone a _drive to create a RAID device [default=/dev/md%s]."
msgstr ""

#: iw/partition_gui.py:1311
msgid "Couldn't Create Drive Clone Editor"
msgstr "ドライブクローンエディターを作成できませんでした"

#: iw/partition_gui.py:1312
msgid "The drive clone editor could not be created for some reason."
msgstr "ドライブクローンエディターは、何らかの理由で作成できませんでした。"

#: iw/partition_gui.py:1346
msgid "Ne_w"
msgstr ""

#: iw/partition_gui.py:1349
msgid "Re_set"
msgstr "リセット(_S)"

#: iw/partition_gui.py:1350
msgid "R_AID"
msgstr ""

#: iw/partition_gui.py:1351
msgid "_LVM"
msgstr ""

#: iw/partition_gui.py:1392
msgid "Hide RAID device/LVM Volume _Group members"
msgstr ""

#: iw/partition_ui_helpers_gui.py:108 iw/partition_ui_helpers_gui.py:130
#: iw/partition_ui_helpers_gui.py:132
msgid "<Not Applicable>"
msgstr "<適用外>"

#: iw/partition_ui_helpers_gui.py:310
msgid "_Format as:"
msgstr "フォーマットする (_F):"

#: iw/partition_ui_helpers_gui.py:332
msgid "Mi_grate filesystem to:"
msgstr "ファイルシステムを移動する:"

#: iw/partition_ui_helpers_gui.py:362
msgid "_Resize"
msgstr "サイズを変更 (_R)"

#: iw/partition_ui_helpers_gui.py:431
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""
"タイプ '%s' のパーティションは単独ドライブに入れる必要があります。これは行う"
"には、「選択可能なドライブ」のチェックリストからそのドライブを選択します。"

#: iw/progress_gui.py:37
#, fuzzy
msgid "Installing Packages"
msgstr "パッケージのインストールでエラー"

#: iw/raid_dialog_gui.py:336
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""
"RAID デバイスを作成するには、少なくとも 2 つの未使用ソフトウェア RAID パー"
"ティションが必要です。\n"
" \n"
"最初に「ソフトウェア RAID」タイプのパーティションを少なくとも2つ作成してか"
"ら、再度「RAID」オプションを選択してください。"

#: iw/raid_dialog_gui.py:350 iw/raid_dialog_gui.py:787
msgid "Make RAID Device"
msgstr "RAID デバイスの作成"

#: iw/raid_dialog_gui.py:353
#, python-format
msgid "Edit RAID Device: %s"
msgstr "RAID デバイスの編集: %s"

#: iw/raid_dialog_gui.py:355
msgid "Edit RAID Device"
msgstr "RAID デバイスの編集"

#: iw/raid_dialog_gui.py:426
msgid "RAID _Device:"
msgstr "RAID デバイス (_D):"

#: iw/raid_dialog_gui.py:444
msgid "RAID _Level:"
msgstr "RAID レベル(_L):"

#: iw/raid_dialog_gui.py:486
msgid "_RAID Members:"
msgstr "RAID メンバー(_R):"

#: iw/raid_dialog_gui.py:503
msgid "Number of _spares:"
msgstr "スペア数(_S):"

#: iw/raid_dialog_gui.py:513
msgid "_Format partition?"
msgstr "パーティションをフォーマットしますか(_F)?"

#: iw/raid_dialog_gui.py:587
msgid ""
"The source drive has no partitions to be cloned.  You must first define "
"partitions of type 'software RAID' on this drive before it can be cloned."
msgstr ""
"ソースドライブにはクローンを作成するパーティションがありません。クローンを作"
"成する前に、このドライブ上で「ソフトウェア RAID」タイプのパーティションを定義"
"する必要があります。"

#: iw/raid_dialog_gui.py:591 iw/raid_dialog_gui.py:597
#: iw/raid_dialog_gui.py:611 iw/raid_dialog_gui.py:624
msgid "Source Drive Error"
msgstr "ソースドライブエラーです"

#: iw/raid_dialog_gui.py:598
msgid ""
"The source drive you selected has partitions which are not of type 'software "
"RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""
"選択されたソースドライブには「ソフトウェア RAID」タイプではないパーティション"
"があります。\n"
"\n"
"これらのパーティションを削除しないと、このドライブのクローンを作成できませ"
"ん。"

#: iw/raid_dialog_gui.py:612
#, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to "
"the drive %s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this "
"drive can be cloned. "
msgstr ""
"選択されたソースドライブにはドライブ %s に入っていないパーティションがありま"
"す。\n"
"\n"
"これらのパーティションを削除するか、このドライブに制限しないと、このドライブ"
"のクローンを作成できません。"

#: iw/raid_dialog_gui.py:625
msgid ""
"The source drive you selected has software RAID partition(s) which are "
"members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""
"選択されたソースドライブにはアクティブなソフトウェア RAID デバイスのメンバー"
"であるソフトウェア RAID パーティションがあります。\n"
"\n"
"これらのパーティションを削除しないと、このドライブのクローンを作成できませ"
"ん。"

#: iw/raid_dialog_gui.py:639 iw/raid_dialog_gui.py:645
#: iw/raid_dialog_gui.py:661
msgid "Target Drive Error"
msgstr "ターゲットドライブのエラー"

#: iw/raid_dialog_gui.py:640
msgid "Please select the target drives for the clone operation."
msgstr "クローン作成を行うターゲットドライブを選択してください。"

#: iw/raid_dialog_gui.py:646
#, python-format
msgid "The source drive %s cannot be selected as a target drive as well."
msgstr "ソースドライブ %s は、同時にターゲットドライブとして選択できません。"

#: iw/raid_dialog_gui.py:662
#, fuzzy, python-format
msgid ""
"The target drive %s has a partition which cannot be removed for the "
"following reason:\n"
"\n"
"\"%s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""
"ターゲットドライブ %(path)s には、次の理由で削除できないパーティションが あり"
"ます:\n"
"\n"
"\"%(rc)s\"\n"
"\n"
"このドライブをターゲットにする前に、このパーティションを削除する必要がありま"
"す。"

#: iw/raid_dialog_gui.py:725
msgid "Please select a source drive."
msgstr "ソースドライブを選択してください。"

#: iw/raid_dialog_gui.py:745
#, python-format
msgid ""
"The drive %s will now be cloned to the following drives:\n"
"\n"
msgstr ""
"ドライブ %s が次のドライブに対してクローン作成されます。\n"
"\n"

#: iw/raid_dialog_gui.py:750
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""
"\n"
"\n"
"警告! ターゲットドライブ上のすべてのデータは消失します。"

#: iw/raid_dialog_gui.py:753
msgid "Final Warning"
msgstr "最終警告"

#: iw/raid_dialog_gui.py:755
msgid "Clone Drives"
msgstr "クローンドライブ"

#: iw/raid_dialog_gui.py:764
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr ""
"ターゲットドライブ内を消去している際にエラーが発生しました。クローン作成は失"
"敗しました。"

#: iw/raid_dialog_gui.py:797
msgid ""
"Clone Drive Tool\n"
"\n"
"This tool allows you to significantly reduce the amount of effort required "
"to setup RAID arrays.  This tool uses a source drive which has been prepared "
"with the desired partitioning layout, and clones this layout onto other "
"similar sized drives.  Then a RAID device can be created.\n"
"\n"
"NOTE: The source drive must have partitions which are restricted to be on "
"that drive only, and can only contain unused software RAID partitions.  "
"Other partition types are not allowed.\n"
"\n"
"EVERYTHING on the target drive(s) will be destroyed by this process."
msgstr ""

#: iw/raid_dialog_gui.py:817
msgid "Source Drive:"
msgstr "ソースドライブ:"

#: iw/raid_dialog_gui.py:825
msgid "Target Drive(s):"
msgstr "ターゲットドライブ:"

#: iw/raid_dialog_gui.py:833
msgid "Drives"
msgstr "ドライブ"

#: iw/task_gui.py:64
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""
"リポジトリーからパッケージのメタデータを読み込めません。repodata ディレクト"
"リーがないことが原因の可能性があります。リポジトリーが正しく生成されているか"
"確認してください。\n"
"\n"
"%s"

#: iw/task_gui.py:141
msgid "Edit Repository"
msgstr "リポジトリーの編集"

#: iw/task_gui.py:165
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""
"リポジトリー %s はすでに追加されています。別のリポジトリー名と URL を選択して"
"ください。"

#: iw/task_gui.py:245
msgid "Invalid Proxy URL"
msgstr "無効なプロキシ URL です。"

#: iw/task_gui.py:246
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr "プロキシには、HTTP, HTTPS, または FTP のURLを入力する必要があります。"

#: iw/task_gui.py:257 iw/task_gui.py:412
msgid "Invalid Repository URL"
msgstr "無効なリポジトリー URL です。"

#: iw/task_gui.py:258 iw/task_gui.py:413
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr ""
"リポジトリーには、HTTP, HTTPS, または FTP の URL を入力する必要があります。"

#: iw/task_gui.py:279 iw/task_gui.py:422
msgid "No Media Found"
msgstr "メディアが見つかりません。"

#: iw/task_gui.py:280 iw/task_gui.py:423
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr ""
"インストール用メディアが見付かりません。ディスクをドライブに挿入して もう一度"
"試して下さい。"

#: iw/task_gui.py:308 iw/task_gui.py:443
msgid "Please enter an NFS server and path."
msgstr "NFS サーバーとパスを入力してください。"

#: iw/task_gui.py:318
#, python-format
msgid ""
"The following error occurred while setting up the repository:\n"
"\n"
"%s"
msgstr ""
"リポジトリーの設定中に次のエラーが見つかりました。\n"
"\n"
"%s"

#: iw/task_gui.py:341
msgid "Invalid Repository Name"
msgstr "無効なリポジトリー名です"

#: iw/task_gui.py:342
msgid "You must provide a repository name."
msgstr "リポジトリー名を入力しなければなりません。"

#: iw/task_gui.py:475 ui/addrepo.glade.h:3
msgid "Add Repository"
msgstr "リポジトリーの追加"

#: iw/task_gui.py:480
msgid "No Software Repos Enabled"
msgstr "ソフトウェアレポジトリが有効になっていません"

#: iw/task_gui.py:481
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr ""
"インストールを続行する為には少なくとも１つのソフトウェアリポジトリーが有効に "
"なっている必要があります。"

#: iw/timezone_gui.py:63 textw/timezone_text.py:95
msgid "Time Zone Selection"
msgstr "タイムゾーンの選択"

#: iw/upgrade_bootloader_gui.py:37 textw/upgrade_bootloader_text.py:136
msgid "Upgrade Boot Loader Configuration"
msgstr "ブートローダー設定のアップグレード"

#: iw/upgrade_bootloader_gui.py:123
msgid "_Update boot loader configuration"
msgstr "ブートローダー設定の更新(_U)"

#: iw/upgrade_bootloader_gui.py:124
msgid "This will update your current boot loader."
msgstr "これで現在のブートローダーを更新します。"

#: iw/upgrade_bootloader_gui.py:128 textw/upgrade_bootloader_text.py:108
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr "システムが変わったので、ブートローダー設定は自動的には更新されません。"

#: iw/upgrade_bootloader_gui.py:131 textw/upgrade_bootloader_text.py:112
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr "インストーラーは現在システムで使用中のブートローダーを検出できません。"

#: iw/upgrade_bootloader_gui.py:138 textw/upgrade_bootloader_text.py:121
#, fuzzy, python-format
msgid ""
"The installer has detected the %s boot loader currently installed on %s."
msgstr ""
"インストーラーが現在 %(bootDev)s にインストールされている %(type)s ブートロー"
"ダーを検出しました。"

#: iw/upgrade_bootloader_gui.py:142
msgid "This is the recommended option."
msgstr "これは推奨されるオプションです。"

#: iw/upgrade_bootloader_gui.py:147
msgid "_Create new boot loader configuration"
msgstr "新しいブートローダー設定を作成(_C)"

#: iw/upgrade_bootloader_gui.py:149
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""
"このオプションは新しいブートローダー設定を作成します。ブートローダーを変更し"
"たい場合は、これを選択してください。"

#: iw/upgrade_bootloader_gui.py:156
msgid "_Skip boot loader updating"
msgstr "ブートローダーの更新をスキップ(_S)"

#: iw/upgrade_bootloader_gui.py:157
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""
"このオプションはブートローダー設定をまったく変更しません。他社製ブートロー"
"ダーを使用している場合は、これを選択してください。"

#: iw/upgrade_bootloader_gui.py:168
msgid "What would you like to do?"
msgstr "どうしますか?"

#: iw/upgrade_migratefs_gui.py:38 textw/upgrade_text.py:36
msgid "Migrate File Systems"
msgstr "ファイルシステムの移行"

#: iw/upgrade_migratefs_gui.py:69
#, fuzzy, python-format
msgid ""
"This release of %s supports an updated file system, which has several "
"benefits over the file system traditionally shipped in %s.  This "
"installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"%(productName)s のこのリリースでは最新のファイルシステムをサポートしていま"
"す。これらは従来 %(productName)s で配布されていたファイルシステムに比べ優れた"
"点をいくつか備えています。このインストールプログラムは、データを失うことなく"
"フォーマットしたパーティションを移行することができます。\n"
"\n"
"これらのどのパーティションを移行しますか?"

#: iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr "swap パーティションのアップグレード"

#: iw/upgrade_swap_gui.py:93 textw/upgrade_text.py:101
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have %"
"dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""
"最近のカーネル(2.4以降)では旧来のカーネルに比べ、膨大な swap 領域を必要とし、"
"最大でシステムの RAM の 2 倍が必要です。現在、 %dMB の swap 領域を確保してい"
"ますが、ここでファイルシステムのいずれかに追加の swap 領域を作成することがで"
"きます。"

#: iw/upgrade_swap_gui.py:100
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""
"\n"
"\n"
"インストーラーは %s MB の RAM を検出しました。\n"

#: iw/upgrade_swap_gui.py:112
msgid "I _want to create a swap file"
msgstr "swap ファイルを作成します(_W)"

#: iw/upgrade_swap_gui.py:121
msgid "Select the _partition to put the swap file on:"
msgstr "swap ファイルを配置するパーティションを選択(_P):"

#: iw/upgrade_swap_gui.py:139 textw/upgrade_text.py:119
msgid "Partition"
msgstr "パーティション"

#: iw/upgrade_swap_gui.py:139
msgid "Free Space (MB)"
msgstr "空き領域 (MB)"

#: iw/upgrade_swap_gui.py:157
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""
"最低でも %d MB の swap ファイルにすることを推奨します。swap ファイルの容量を"
"入力してください。"

#: iw/upgrade_swap_gui.py:172
msgid "Swap file _size (MB):"
msgstr " swap ファイルの容量 (MB)(_S):"

#: iw/upgrade_swap_gui.py:182
msgid "I _don't want to create a swap file"
msgstr "swap ファイルを作成しません(_D)"

#: iw/upgrade_swap_gui.py:192
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""
"swap ファイルの作成を強く推奨します。 swap ファイルを作成しないと、インストー"
"ラーが異常終了する原因となる場合があります。本当に続行しますか?"

#: iw/upgrade_swap_gui.py:200 textw/upgrade_text.py:188
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr "swap ファイルは 1 MB から 2000 MB の間の容量にしてください。"

#: iw/upgrade_swap_gui.py:207 textw/upgrade_text.py:183
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr "swap パーティション用に選択したデバイスには十分な領域がありません。"

#: iw/welcome_gui.py:56 textw/welcome_text.py:36
msgid "Network Install Required"
msgstr "ネットワークのインストールが必要"

#: iw/welcome_gui.py:57 textw/welcome_text.py:37
msgid ""
"Your installation source is set to a network location, but no netork devices "
"were found on your system.  To avoid a network installation, boot with the "
"full DVD, full CD set, or do not pass a repo= parameter that specifies a "
"network source."
msgstr ""
"インストールのソースがネットワークの位置に設定してありますが、システム上に "
"ネットワークデバイスが見当たりません。ネットワークインストールを避けるには、 "
"完全な DVD か完全な CD セットで起動しますが、その場合、ネットワークソースを "
"指定してしまう repo= parameter を押さないで下さい。"

#: iw/welcome_gui.py:67
msgid "E_xit Installer"
msgstr "インストーラーを終了(_X)"

#: iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr "z/IPL ブートローダーの設定"

#: iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr "z/IPL ブートローダーがシステムにインストールされます。"

#: iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""
"z/IPL ブートローダーがシステムにインストールされます。\n"
"\n"
"root パーティションはパーティションの設定で既に選択したパーティションになりま"
"す。\n"
"\n"
"マシンの起動に使用されるカーネルはデフォルトでインストールされるカーネルにな"
"ります。\n"
"\n"
"インストール後に変更を加える場合は、/etc/zipl.conf の設定ファイルを自由に変更"
"できます。\n"
"\n"
"ここでご使用のマシンまたは設定が必要とする追加のカーネルパラメーターを入力す"
"ることができます。"

#: iw/zipl_gui.py:90 textw/zipl_text.py:71
msgid "Kernel Parameters"
msgstr "カーネルパラメーター"

#: iw/zipl_gui.py:93 iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr "Chandev パラメーター"

#: loader/cdinstall.c:185 loader/cdinstall.c:206 loader/mediacheck.c:60
msgid "Media Check"
msgstr "メディアチェック"

#: loader/cdinstall.c:185 loader/cdinstall.c:188 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/method.c:326
msgid "Test"
msgstr "テスト"

#: loader/cdinstall.c:185 loader/cdinstall.c:189
msgid "Eject Disc"
msgstr "ディスク取り出し"

#: loader/cdinstall.c:186
#, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""
"現在ドライブに入っているディスクをテストするには \"%s\" を選び、 ディスクを取"
"り出して別のディスクを挿入してテストをするには。 \"%s\" を選びます。"

#: loader/cdinstall.c:207
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""
"追加のメディアをテストする場合は、次のディスクを挿入してから \"%s\" を押して"
"ください。それぞれのディスクのテストは絶対に必要なわけではありませんが、強く"
"推奨します。最低でも、はじめて使用する前にはテストを行ってください。テストが"
"成功したら、再度使用する前に各ディスクをテストする必要はありません。"

#: loader/cdinstall.c:229
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""
"いずれのドライブにも %s ディスク は見つかりませんでした。%s ディスクを挿入し"
"てから、%s を押して再試行してください。"

#: loader/cdinstall.c:248
msgid "Disc Found"
msgstr "ディスク が見つかりました"

#: loader/cdinstall.c:249
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""
"インストールの前にメディアのテストを開始するには %s を押します。\n"
"\n"
"メディアテストをスキップするには %s を選択して、インストールを開始します。"

#: loader/cdinstall.c:339
msgid "Scanning"
msgstr "スキャン中"

#: loader/cdinstall.c:339
#, c-format
msgid "Looking for installation images on CD device %s\n"
msgstr "CD デバイス %s 上のインストールイメージを探しています\n"

#: loader/cdinstall.c:341
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr "CD デバイス %s 上のインストールイメージを探しています。"

#: loader/cdinstall.c:442
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""
"いずれの CDROM ドライブにも %s ディスクは見つかりませんでした。%s ディスクを"
"挿入してから、%s を押して再試行してください。"

#: loader/cdinstall.c:451
msgid "Disc Not Found"
msgstr "ディスクが見つかりません"

#: loader/cdinstall.c:452 loader/driverdisk.c:244 loader/driverdisk.c:275
#: loader/driverdisk.c:359 loader/driverselect.c:81 loader/driverselect.c:179
#: loader/driverselect.c:205 loader/hdinstall.c:222 loader/hdinstall.c:279
#: loader/kbd.c:119 loader/loader.c:483 loader/loader.c:503
#: loader/loader.c:1375 loader/loader.c:1408 loader/net.c:582 loader/net.c:953
#: loader/net.c:1880 loader/net.c:1902 loader/nfsinstall.c:91
#: loader/urls.c:292 storage/__init__.py:141 storage/devicetree.py:85
#: textw/constants_text.py:52
msgid "Back"
msgstr "戻る"

#: loader/cdinstall.c:518
msgid "Cannot find kickstart file on CDROM."
msgstr "ディスクROM 上にキックスタートファイルが見つかりません。"

#: loader/copy.c:51 loader/method.c:280
#, c-format
msgid "Failed to read directory %s: %m"
msgstr "%s ディレクトリーの読み込みに失敗: %m"

#: loader/driverdisk.c:139
msgid "Loading"
msgstr "読み込み中"

#: loader/driverdisk.c:139
msgid "Reading driver disk"
msgstr "ドライバーディスクを読み込み中"

#: loader/driverdisk.c:238 loader/driverdisk.c:270
msgid "Driver Disk Source"
msgstr "ドライバーディスクソース"

#: loader/driverdisk.c:239
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""
"ドライバーディスク用のソースとして使用できるデバイスが複数あります。使用する"
"デバイスを指定してください。"

#: loader/driverdisk.c:271
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""
"このデバイスにはドライバーディスクのイメージを格納できるパーティションが複数"
"あります。使用するパーティションを指定してください。"

#: loader/driverdisk.c:306
msgid "Failed to mount partition."
msgstr "パーティションのマウントに失敗しました。"

#: loader/driverdisk.c:311
msgid "Select driver disk image"
msgstr "ドライバーディスクイメージを選択してください"

#: loader/driverdisk.c:312
msgid "Select the file which is your driver disk image."
msgstr "ドライバーディスクイメージのファイルを選択してください。"

#: loader/driverdisk.c:341
msgid "Failed to load driver disk from file."
msgstr "ファイルからのドライバーディスク読み込みに失敗しました。"

#: loader/driverdisk.c:353
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr "ドライバーディスクを /dev/%s に挿入し、「OK」 を押して続行します。"

#: loader/driverdisk.c:359
msgid "Insert Driver Disk"
msgstr "ドライバーディスクを挿入"

#: loader/driverdisk.c:372
msgid "Failed to mount driver disk."
msgstr "ドライバーディスクのマウントに失敗しました。"

#: loader/driverdisk.c:380
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr "この %s のリリースにはドライバーディスクは無効です。"

#: loader/driverdisk.c:443
msgid "Manually choose"
msgstr "手動で選択"

#: loader/driverdisk.c:444
msgid "Load another disk"
msgstr "別のディスクをロード"

#: loader/driverdisk.c:445
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""
"このドライバーディスク上では適切なタイプのデバイスが見つかりませんでした。手"
"動でドライバーを選択する、このまま続行する、別のドライバーディスクを読み込"
"む、のいずれかを指定してください。"

#: loader/driverdisk.c:483
msgid "Driver disk"
msgstr "ドライバーディスク"

#: loader/driverdisk.c:484
msgid "Do you have a driver disk?"
msgstr "ドライバーディスクがありますか?"

#: loader/driverdisk.c:493
msgid "More Driver Disks?"
msgstr "他のドライバーディスク?"

#: loader/driverdisk.c:494
msgid "Do you wish to load any more driver disks?"
msgstr "他のドライバーディスクを読み込みますか?"

#: loader/driverdisk.c:539 loader/driverdisk.c:576 loader/hdinstall.c:372
#: loader/kickstart.c:132 loader/kickstart.c:142 loader/kickstart.c:185
#: loader/kickstart.c:563 loader/modules.c:364 loader/modules.c:376
#: loader/net.c:1587 loader/net.c:1608 loader/nfsinstall.c:337
#: loader/urlinstall.c:451 loader/urlinstall.c:460 loader/urlinstall.c:467
msgid "Kickstart Error"
msgstr "キックスタートエラー"

#: loader/driverdisk.c:540
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr "キックスタートの元となるドライバーディスクが不明です: %s"

#: loader/driverdisk.c:577
#, fuzzy, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s:%s"
msgstr ""
"キックスタートのドライバーディスクコマンドに以下のような不正な引数が指定され"
"ました: %s"

#: loader/driverselect.c:67
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""
"%s モジュールに渡すパラメーターを空白で区切って入力してください。どのパラメー"
"ターを使用するのかわからない場合、「OK」 ボタンを押してこの画面をスキップして"
"ください。"

#: loader/driverselect.c:91
msgid "Enter Module Parameters"
msgstr "モジュールパラメーターを入力"

#: loader/driverselect.c:178
msgid "No drivers found"
msgstr "ドライバーが見つかりません"

#: loader/driverselect.c:178
msgid "Load driver disk"
msgstr "ドライバーディスクの読み込み"

#: loader/driverselect.c:179
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""
"手動で挿入するドライバーが見つかりませんでした。ドライバーディスクを使用しま"
"すか?"

#: loader/driverselect.c:197
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""
"読み込むドライバーを下記から選択してください。必要なドライバーが表示されてい"
"ないがドライバーディスクがある場合、F2 を押してください。"

#: loader/driverselect.c:206
msgid "Specify optional module arguments"
msgstr "オプションのモジュール引数を指定してください"

#: loader/driverselect.c:234
msgid "Select Device Driver to Load"
msgstr "読み込むデバイスドライバーを選択してください"

#: loader/hdinstall.c:135
msgid ""
"An error occured finding the installation image on your hard drive.  Please "
"check your images and try again."
msgstr ""
"ハードドライブからインストールイメージを読み込み中にエラーが発生しました。 ご"
"使用のイメージを確認してからやり直してください。"

#: loader/hdinstall.c:223
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""
"このシステムにはハードドライブがないようです。追加デバイスの設定を行いますか?"

#: loader/hdinstall.c:236
#, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""
"%s 用 インストールイメージを格納しているパーティションとディレクトリーを指定"
"してください。一覧内に、使用しているディスクドライブがなければ、 F2 を押して"
"追加デバイスの設定を行ってください。"

#: loader/hdinstall.c:263
msgid "Directory holding image:"
msgstr "イメージを格納しているディレクトリー:"

#: loader/hdinstall.c:291
msgid "Select Partition"
msgstr "パーティションを選択してください"

#: loader/hdinstall.c:341
#, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr "デバイス %s にはインストールイメージが含まれていないようです。"

#: loader/hdinstall.c:373
#, fuzzy, c-format
msgid "Bad argument to HD kickstart method command %s: %s"
msgstr "HD キックスタートコマンド に対する引数が不正です: %s"

#: loader/hdinstall.c:443 loader/hdinstall.c:499
msgid "Cannot find kickstart file on hard drive."
msgstr "ハードドライブにキックスタートファイルが見つかりません。"

#: loader/hdinstall.c:486
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr "BIOS ディスク %s 用のハードドライブが見つかりません"

#: loader/kbd.c:117
msgid "Keyboard Type"
msgstr "キーボードの種類"

#: loader/kbd.c:118
msgid "What type of keyboard do you have?"
msgstr "ご使用のキーボードの種類を指定してください。"

#: loader/kickstart.c:133
#, c-format
msgid "Error opening kickstart file %s: %m"
msgstr "キックスタートファイル %s を開く際にエラー: %m"

#: loader/kickstart.c:143
#, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr "キックスタートファイル %s の読み込み時にエラー: %m"

#: loader/kickstart.c:186
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr "%s のエラーが %d 行目(キックスタートファイル %s の)で発生しました。"

#: loader/kickstart.c:285
msgid "Cannot find ks.cfg on removable media."
msgstr "リムーバブルメディアに ks.cfg が見つかりません。"

#: loader/kickstart.c:387
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""
"キックスタートファイルをダウンロードすることができません。以下のキックスター"
"トパラメーターを変更するか、対話インストールとして続行するために Cancel を押"
"してください。"

#: loader/kickstart.c:396
msgid "Error downloading kickstart file"
msgstr "キックスタートファイルをダウンロードする際にエラー発生"

#: loader/kickstart.c:564
#, fuzzy, c-format
msgid "Bad argument to shutdown kickstart method command %s: %s"
msgstr "シャットダウンキックスタートメソッドコマンドに対する引数が不正です: %s"

#: loader/lang.c:64
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "%s (%s 用)へようこそ - レスキューモード"

#: loader/lang.c:65 loader/loader.c:226
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr "  <Tab>/<Alt-Tab> 項目間の移動 | <Space> 選択 | <F12> 次の画面 "

#: loader/lang.c:376
msgid "Choose a Language"
msgstr "言語を選択してください"

#: loader/loader.c:127
msgid "Local CD/DVD"
msgstr "ローカル CD/DVD"

#: loader/loader.c:128
msgid "Hard drive"
msgstr "ハードドライブ"

#: loader/loader.c:129
msgid "NFS directory"
msgstr "NFS ディレクトリー:"

#: loader/loader.c:434 loader/loader.c:478
msgid "Update Disk Source"
msgstr "ディスクソースの更新"

#: loader/loader.c:435
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""
"更新ディスク用のソースとして使用できるデバイスが複数あります。使用するデバイ"
"スを指定してください。"

#: loader/loader.c:479
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""
"このデバイスにはアップデートディスクイメージを含んでいるかもしれないパーティ"
"ションが複数あります。使用するパーティションを指定してください。"

#: loader/loader.c:497
#, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr "更新ディスクを %s に挿入してから 「OK」 を押して続行します。"

#: loader/loader.c:503
msgid "Updates Disk"
msgstr "更新ディスク"

#: loader/loader.c:521
msgid "Failed to mount updates disk"
msgstr "更新ディスクのマウントに失敗しました"

#: loader/loader.c:526
msgid "Updates"
msgstr "更新"

#: loader/loader.c:526
msgid "Reading anaconda updates"
msgstr "anaconda の更新を読み込み中"

#: loader/loader.c:562
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""
"アップデートイメージをダウンロードできません。以下のアップデートの場所を編集"
"するか、キャンセルを押してアップデートを中止してください。"

#: loader/loader.c:571
msgid "Error downloading updates image"
msgstr "アップデートイメージのダウンロード中にエラー発生"

#: loader/loader.c:1184
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr "このマシンには %s のインストールに十分な RAM がありません。"

#: loader/loader.c:1241
msgid "Media Detected"
msgstr "検出されたメディア"

#: loader/loader.c:1242
msgid "Found local installation media"
msgstr "ローカルのインストールメディアが検出されました"

#: loader/loader.c:1367
msgid "Rescue Method"
msgstr "レスキュー方法"

#: loader/loader.c:1368
msgid "Installation Method"
msgstr "インストール方法"

#: loader/loader.c:1370
msgid "What type of media contains the rescue image?"
msgstr "レスキューイメージが格納されているメディアタイプを指定してください。"

#: loader/loader.c:1372
msgid "What type of media contains the installation image?"
msgstr "インストールイメージが格納されているメディアタイプを指定してください。"

#: loader/loader.c:1407
msgid "No driver found"
msgstr "ドライバーが見つかりません"

#: loader/loader.c:1407
msgid "Select driver"
msgstr "ドライバーを選択してください"

#: loader/loader.c:1408
msgid "Use a driver disk"
msgstr "ドライバーディスクを使用する"

#: loader/loader.c:1409
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""
"このインストールタイプに必要な種類のデバイスを見つけることができません。  手"
"動でドライバーを選択しますか、またはドライバーディスクを使用しますか?"

#: loader/loader.c:1639
msgid "The following devices have been found on your system."
msgstr "システム上で次のデバイスが検出されました。"

#: loader/loader.c:1641
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""
"デバイスドライバーはシステムにロードされていません。ドライバーをロードします"
"か?"

#: loader/loader.c:1645
msgid "Devices"
msgstr "デバイス"

#: loader/loader.c:1646
msgid "Done"
msgstr "終了"

#: loader/loader.c:1647
msgid "Add Device"
msgstr "デバイスの追加"

#: loader/loader.c:1859
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr "ローダーはすでに実行されています。シェルの起動中です。\n"

#: loader/loader.c:2241
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr ""
"anaconda %s を %s レスキューモードで実行中 - しばらくお待ちください。\n"

#: loader/loader.c:2243
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr ""
"anaconda %s・%s システムインストーラーを実行中 - しばらくお待ちください。\n"

#: loader/mediacheck.c:46
#, c-format
msgid "Unable to find install image %s"
msgstr "インストールイメージ %s を検出できません。"

#: loader/mediacheck.c:52
#, c-format
msgid "Checking \"%s\"."
msgstr "\"%s\" のチェック中。"

#: loader/mediacheck.c:54
#, c-format
msgid "Checking media."
msgstr "メディアをチェック中。"

#: loader/mediacheck.c:86
msgid ""
"Unable to read the disc checksum from the primary volume descriptor.  This "
"probably means the disc was created without adding the checksum."
msgstr ""
"プライマリボリュームディスクリプターからディスクチェックサムを読み込めませ"
"ん。 恐らく、このディスクはチェックサムを追加しないで作成されています。"

#: loader/mediacheck.c:93
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""
"今テストしたイメージはエラーを含んでいます。欠陥のあるダウンロードまたはディ"
"スク不良が原因の可能性があります。可能ならディスクを清掃してからやり直してく"
"ださい。このテストが失敗を繰り返す場合、インストールは中止すべきです。"

#: loader/mediacheck.c:101
msgid "Success"
msgstr "成功"

#: loader/mediacheck.c:102
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""
"テストしたイメージの検証に成功しました。このメディアからのインストールは問題"
"ないでしょう。ただし、すべてのメディア・ドライブのエラーがメディアチェックで"
"発見できるとは限りません。"

#: loader/method.c:323
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""
"ISO イメージのチェックサムテストを実施しますか: \n"
"\n"
"   %s?"

#: loader/method.c:326
msgid "Checksum Test"
msgstr "チェックサムテスト"

#: loader/modules.c:365
#, fuzzy, c-format
msgid "Bad argument to device kickstart method command %s: %s"
msgstr "デバイスキックスタートメソッドのコマンドに対する引数が不正です: %s"

#: loader/modules.c:377
msgid "A module name must be specified for the kickstart device command."
msgstr "モジュール名がキックスタートデバイスコマンドに指定されていません。"

#: loader/net.c:111
msgid "Invalid Prefix"
msgstr "無効なプレフィックス"

#: loader/net.c:112
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""
"プレフィックスは、IPv4 ネットワークなら 1 から 32 まで、IPv6 ネットワークな"
"ら 1 から 128 までの数値にしなければなりません。"

#: loader/net.c:465 loader/net.c:473 loader/net.c:529
#, c-format
msgid "There was an error configuring your network interface."
msgstr "ネットワークインターフェースの設定中にエラーが発生しました。"

#: loader/net.c:467
#, c-format
msgid ""
"\n"
"This cannot be corrected in cmdline mode.\n"
"Halting.\n"
msgstr ""
"\n"
"これは cmdline モードでは修正できません。\n"
"一時停止します。\n"

#: loader/net.c:472 loader/net.c:528
msgid "Network Error"
msgstr "ネットワークエラー"

#: loader/net.c:560 textw/netconfig_text.py:131
msgid "Enable IPv4 support"
msgstr "IPv4 サポートを有効にする"

#: loader/net.c:574
msgid "Enable IPv6 support"
msgstr "IPv6 サポートを有効にする"

#: loader/net.c:614
msgid "Configure TCP/IP"
msgstr "TCP/IP の設定"

#: loader/net.c:671
msgid "Missing Protocol"
msgstr "見つからないプロトコル"

#: loader/net.c:672
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr "少くともプロトコルを 1 つ選択しなければなりません (IPv4 または IPv6)。"

#: loader/net.c:679
msgid "IPv4 Needed for NFS"
msgstr "NFS には IPv4 が必要です"

#: loader/net.c:680
msgid "NFS installation method requires IPv4 support."
msgstr "NFS インストールを行うには IPv4 のサポートが必要になります。"

#: loader/net.c:779
msgid "IPv4 address:"
msgstr "IPv4 アドレス:"

#: loader/net.c:791 loader/net.c:858 ui/netconfig.glade.h:1
msgid "/"
msgstr "/"

#: loader/net.c:846
msgid "IPv6 address:"
msgstr "IPv6 アドレス:"

#: loader/net.c:912 textw/netconfig_text.py:158
msgid "Gateway:"
msgstr "ゲートウェイ:"

#: loader/net.c:920
msgid "Name Server:"
msgstr "ネームサーバー:"

#: loader/net.c:959
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""
"IPv4 および/または IPv6 アドレスとプレフィックス (アドレス / プレフィックス) "
"を入力してください。 IPv4 では 4 つの数字をドットで区切ったネットマスクか "
"CIDR 形式のプレフィックスが可能です。ゲートウェイとネームサーバーのフィールド"
"は、必ず有効な IPv4 または IPv6 アドレスでなければなりません。"

#: loader/net.c:979
msgid "Manual TCP/IP Configuration"
msgstr "手動による TCP/IP 設定"

#: loader/net.c:1102 loader/net.c:1109
msgid "Missing Information"
msgstr "情報が不足しています"

#: loader/net.c:1103
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""
"有効な IPv4 アドレスとネットマスクまたは CIDR プレフィックスの両方を入力する"
"必要があります。"

#: loader/net.c:1110
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr ""
"有効な IPv6 アドレスと CIDR プレフィックスの両方を入力する必要があります。"

#: loader/net.c:1588
#, fuzzy, c-format
msgid "Bad argument to kickstart network command %s: %s"
msgstr "キックスタートネットワークコマンドに対する引数が不正です: %s"

#: loader/net.c:1609
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr "ネットワークコマンドで不正な bootproto %s が指定されました"

#: loader/net.c:1686
msgid "Seconds:"
msgstr "2 番目:"

#: loader/net.c:1875
msgid "Networking Device"
msgstr "ネットワーク構築デバイス"

#: loader/net.c:1876
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""
"このシステムには複数のネットワークデバイスが存在します。インストールの際に使"
"用するデバイスを選択してください。"

#: loader/net.c:1880
msgid "Identify"
msgstr "識別"

#: loader/net.c:1889
msgid "You can identify the physical port for"
msgstr "以下の為の物理ポートを識別できます"

#: loader/net.c:1891
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""
"数秒間だけ LED の光源を点滅することで達成します。１から30までの数字を挿入し"
"て LED ポートの光源を点滅する時間をセットします。"

#: loader/net.c:1901
msgid "Identify NIC"
msgstr "NIC の識別"

#: loader/net.c:1914
msgid "Invalid Duration"
msgstr "無効な期間"

#: loader/net.c:1915
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr "１から30までの整数で秒数を入力する必要があります。"

#: loader/net.c:1927
#, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr "%s ポートの光源を %d 秒だけ点滅しています。"

#: loader/net.c:2096 loader/net.c:2100
#, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr "NetworkManager の %s の設定を待っています。\n"

#: loader/nfsinstall.c:68
msgid "NFS server name:"
msgstr "NFS サーバー名:"

#: loader/nfsinstall.c:72
#, c-format
msgid "%s directory:"
msgstr "%s ディレクトリー:"

#: loader/nfsinstall.c:83
#, fuzzy, c-format
msgid "Please enter the server name and path to your %s installation image."
msgstr ""
"%s インストールイメージへの NFSv3 パスとサーバー名を入力して、オプションとし"
"て 追加の NFS マウントポイントを入力して下さい。"

#: loader/nfsinstall.c:90
msgid "NFS Setup"
msgstr "NFS 設定"

#: loader/nfsinstall.c:251
msgid "That directory could not be mounted from the server."
msgstr "そのディレクトリーはサーバーからマウントできませんでした。"

#: loader/nfsinstall.c:262
#, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr "そのディレクトリーは %s のインストールイメージを含んでいないようです。"

#: loader/nfsinstall.c:338
#, fuzzy, c-format
msgid "Bad argument to NFS kickstart method command %s: %s"
msgstr "NFS キックスタートメソッドコマンドに対する引数が不正です: %s"

#: loader/telnetd.c:89 loader/telnetd.c:128
msgid "Telnet"
msgstr "Telnet"

#: loader/telnetd.c:89
msgid "Waiting for telnet connection."
msgstr "telnet 接続の待機中。"

#: loader/telnetd.c:128
msgid "Running anaconda via telnet."
msgstr "telnet 経由で anaconda を実行中。"

#: loader/urlinstall.c:84
#, fuzzy, c-format
msgid "Unable to retrieve %s://%s%s."
msgstr "%s の取り込みが出来ません。"

#: loader/urlinstall.c:176
msgid "Unable to retrieve the install image."
msgstr "インストールイメージを取り込むことができません。"

#: loader/urlinstall.c:452
#, fuzzy, c-format
msgid "Bad argument to Url kickstart method command %s: %s"
msgstr "URL キックスタートメソッドコマンドに対する引数が不正です: %s"

#: loader/urlinstall.c:461
msgid "Must supply a --url argument to Url kickstart method."
msgstr "Url キックスタート方法には、 --url の引数を使用する必要があります。"

#: loader/urlinstall.c:468
#, c-format
msgid "Unknown Url method %s"
msgstr "%s は不明な Url 方法です"

#: loader/urls.c:235 loader/urls.c:237 loader/urls.c:240
msgid "Retrieving"
msgstr "取り込み中"

#: loader/urls.c:295
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr ""
"%s のインストールイメージを含んでいるサーバーの URL を入力してください。"

#: loader/urls.c:321
msgid "URL Setup"
msgstr "URLの設定"

#: loader/urls.c:329
msgid "You must enter a URL."
msgstr "URLを入力してください。"

#: loader/urls.c:335
msgid "URL must be either an ftp or http URL"
msgstr "URL は ftp か http のものである必要があります。"

#: loader/urls.c:346
#, fuzzy
msgid "Unknown Host"
msgstr "不明な Linux システム"

#: loader/urls.c:347
#, fuzzy, c-format
msgid "%s is not a valid hostname."
msgstr "%s は有効な IPv6 アドレスではありません。"

#: loader/windows.c:65
msgid "Loading SCSI driver"
msgstr "SCSI ドライバーを読み込み中"

#: loader/windows.c:66
#, c-format
msgid "Loading %s driver"
msgstr "%s ドライバーを読み込み中。"

#: storage/__init__.py:92
msgid "Unknown Device"
msgstr "不明なデバイス"

#: storage/__init__.py:93
#, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""
"デバイス %s で与えられたインストールソースが見付かりません。パラメータを"
"チェックして もう一度試して下さい。"

#: storage/__init__.py:104
msgid "Installation cannot continue."
msgstr "インストールを続行できません。"

#: storage/__init__.py:105
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""
"選択されたストレージ設定は既にアクティベートされています。ここからディスク編"
"集画面に戻ることはできません。インストールプロセスを続けますか?"

#: storage/__init__.py:133
msgid "Encrypt device?"
msgstr "デバイスの暗号化？"

#: storage/__init__.py:134
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""
"暗号化が有効になっているはずのブロックデバイスを指定しています、しかし、パス"
"フレーズが供給されていません。戻ってパスフレーズを記入しないとブロックデバイ"
"スの暗号化は無効になります。"

#: storage/__init__.py:157
msgid "Writing storage configuration to disk"
msgstr "ストレージ設定をディスクに書き込み中"

#: storage/__init__.py:158
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost."
msgstr ""
"選択したディスク分割のオプションを書き込みます。削除及び再フォーマットする "
"パーティションにあるデータはすべて失われます。"

#: storage/__init__.py:163
msgid "Go _back"
msgstr "戻る(_B)"

#: storage/__init__.py:164
msgid "_Write changes to disk"
msgstr "変更をディスクに書き込む (_W)"

#: storage/__init__.py:184
msgid "Running..."
msgstr "実行中..."

#: storage/__init__.py:185
msgid "Storing encryption keys"
msgstr "暗号化キーを格納"

#: storage/__init__.py:201
#, python-format
msgid "Error storing an encryption key: %s\n"
msgstr "暗号化キーを格納中にエラー: %s\n"

#: storage/__init__.py:318
msgid "Finding Devices"
msgstr "デバイスを検索中"

#: storage/__init__.py:319
msgid "Finding storage devices"
msgstr "ストレージデバイスを検索中"

#: storage/__init__.py:560
msgid "This partition is holding the data for the hard drive install."
msgstr ""
"このパーティションはハードドライブインストール用のデータを保有しています。"

#: storage/__init__.py:565
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr "DASD 形式でフォーマットされた LDL パーティションは削除できません。"

#: storage/__init__.py:571
#, python-format
msgid "This device is part of the RAID device %s."
msgstr "このデバイスは RAID デバイス %s の一部です。"

#: storage/__init__.py:574
msgid "This device is part of a RAID device."
msgstr "このデバイスは RAID デバイスの一部です。"

#: storage/__init__.py:579
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr "このデバイスは LVM ボリュームグループ %s の一部です。"

#: storage/__init__.py:582
msgid "This device is part of a LVM volume group."
msgstr "このデバイスは LVM ボリュームグループの一部です。"

#: storage/__init__.py:598
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""
"このデバイスは拡張パーティションです。この中の論理パーティションは先に 削除で"
"きません:\n"
"\n"

#: storage/__init__.py:870
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"%s のインストールを継続するために必要な root パーティション (/) が定義されて"
"いません。"

#: storage/__init__.py:875
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"root パーティションが 250 MB 以下になっています。通常、 %s のインストールには"
"小さすぎます。"

#: storage/__init__.py:881
#, fuzzy, python-format
msgid ""
"Your / partition is less than %s MB which is lower than recommended for a "
"normal %s install."
msgstr ""
"/  パーティションは %(min)s MB 以下であり、これは通常の %(productName)s イン"
"ストールに推奨できるサイズに足りません。"

#: storage/__init__.py:891
#, python-format
msgid ""
"Your / partition does not match the the live image you are installing from.  "
"It must be formatted as %s."
msgstr ""
"あなたの / パーティションはインストールしようとしているライブイメージと一致し"
"ません。%s としてフォーマットされている必要があります。"

#: storage/__init__.py:898
#, fuzzy, python-format
msgid ""
"Your %s partition is less than %s megabytes which is lower than recommended "
"for a normal %s install."
msgstr ""
"%(mount)s パーティションの容量が %(size)s MB以下であり、通常の %(productName)"
"s のインストールでの 推奨容量には不足しています。"

#: storage/__init__.py:925
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""
"USB デバイス上にインストールしています。稼動するシステムが作成されることもあ"
"りますが、されないこともあります。"

#: storage/__init__.py:928
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""
"FireWire デバイス上にインストールしています。稼動するシステムが作成されること"
"もありますが、されないこともあります。"

#: storage/__init__.py:935
msgid ""
"You have not specified a swap partition.  Due to the amount of memory "
"present, a swap partition is required to complete installation."
msgstr ""
"swap パーティションが未指定です。現在のメモリー量では、インストールを完了させ"
"るには swap パーティションが必要です。"

#: storage/__init__.py:940
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"swap パーティションが未指定です。すべての場合において必ず必要というわけではあ"
"りませんが、ほとんどのインストールにおいて明らかに性能を向上させます。"

#: storage/__init__.py:947
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""
"このマウントポイントは正しくありません。%s ディレクトリーは / ファイルシステ"
"ム上になければなりません。"

#: storage/__init__.py:951
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "マウントポイント %s は linux ファイルシステム上にある必要があります。"

#: storage/__init__.py:962
msgid "No Drives Found"
msgstr "ドライブがみつかりませんでした。"

#: storage/__init__.py:963
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""
"エラー発生 - 新規ファイルシステム作成のための有効なデバイスが見つかりません。"
"ハードウェアをチェックして、この問題の原因を調査してください。"

#: storage/__init__.py:1195 storage/__init__.py:1204
msgid "Dirty File Systems"
msgstr "損傷ファイルシステム"

#: storage/__init__.py:1196
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""
"ご使用の Linux システム用の次のファイルシステムは正常にアンマウントされません"
"でした。アップグレードするには、 Linux インストールを起動し、ファイルシステム"
"のチェックをしてから正常なシャットダウンを行ってください。\n"
"%s"

#: storage/__init__.py:1205
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""
"ご使用の Linux システム用の次のファイルシステムは正常にアンマウントされません"
"でした。とにかくそれらをマウントしますか?\n"
"%s"

#: storage/__init__.py:1677
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for "
"swap space, you must reformat as a new-style Linux swap partition."
msgstr ""
"スワップデバイス:\n"
"\n"
"     %s\n"
"\n"
"は旧式の Linux swap パーティションです。スワップ領域用にこのデバイスを 使用し"
"たい場合は、新スタイルの Linux swap パーティションとして再フォーマットする必"
"要があります。"

#: storage/__init__.py:1688
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""
"/etc/fstab ファイル内の swap デバイス:\n"
"\n"
"      %s\n"
"\n"
"は現在、ソフトウェアサスペンドデバイスとして使用されており、これはシステムが"
"ハイバネート状態であることを意味します。アップグレードを実行するには、システ"
"ムをハイバネートするのではなく、シャットダウンしてください。"

#: storage/__init__.py:1696
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""
"/etc/fstab ファイル内の swap デバイス:\n"
"\n"
"     %s\n"
"\n"
"は現在、ソフトウェアサスペンドデバイスとして使用されており、これはシステムが"
"ハイバネート状態であることを意味します。新規のインストールを実行している場合"
"は、インストーラーがすべての swap パーティションをフォーマットするよう設定さ"
"れていることを確認してください。"

#: storage/__init__.py:1708
#, fuzzy, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a support swap volume.  In order to continue installation, "
"you will need to format the device or skip it."
msgstr ""
"スワップデバイス:\n"
"\n"
"     %s\n"
"\n"
"はサポートされたスワップボリュームを含みません。インストールを継続するには、 "
"デバイスをフォーマットするか、又はスキップします。"

#: storage/__init__.py:1719
#, fuzzy, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""
"スワップデバイス %(name)s を有効にしている際にエラー発生: %(msg)s\n"
"\n"
"アップグレードパーティションの /etc/fstab が有効なスワップデバイスを参照して"
"いません。\n"
"\n"
"OK を押してインストーラを終了してください。"

#: storage/__init__.py:1725
#, fuzzy, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"スワップデバイス %(name)s を有効にしている際にエラー発生: %(msg)s\n"
"\n"
"このスワップデバイスが初期化されていない可能性があります。\n"
"\n"
"OK を押してインストーラを終了してください。"

#: storage/__init__.py:1785 storage/__init__.py:1795
msgid "Invalid mount point"
msgstr "無効なマウントポイント"

#: storage/__init__.py:1786
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%s を作成中にエラーが発生しました。このパスのどこかがディレクトリーではありま"
"せん。致命的なエラーとなるためインストールを続行できません。\n"
"\n"
"<Enter> を押してインストーラを終了してください。"

#: storage/__init__.py:1796
#, fuzzy, python-format
msgid ""
"An error occurred when trying to create %s: %s.  This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"%(mountpoint)s を作成中にエラーが発生しました: %(msg)s。致命的な エラーとなる"
"ためインストールを続行できません。\n"
"\n"
"<Enter> を押してインストーラを終了してください。"

#: storage/__init__.py:1809 storage/__init__.py:1830
msgid "Unable to mount filesystem"
msgstr "ファイルシステムをマウントできません"

#: storage/__init__.py:1810
#, fuzzy, python-format
msgid ""
"An error occurred mounting device %s as %s.  You may continue installation, "
"but there may be problems."
msgstr ""
"デバイス %(path)s を %(mountpoint)s としてマウントする際にエラーが発生しまし"
"た。 インストールは続行できますが、問題が生じる場合があります。"

#: storage/__init__.py:1831
#, fuzzy, python-format
msgid ""
"An error occurred mounting device %s as %s: %s. This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"デバイス %(path)s を %(mountpoint)s としてマウント中にエラーが発生しました: %"
"(msg)s。 致命的なエラーとなるためインストールを続行できません。\n"
"\n"
"<Enter> を押してインストーラを終了してください。"

#: storage/devicelibs/lvm.py:319
#, python-format
msgid "vginfo failed for %s"
msgstr "%s 用の vginfo に失敗"

#: storage/devicelibs/lvm.py:348
#, python-format
msgid "lvs failed for %s"
msgstr "%s 用の lvs に失敗"

#: storage/devicetree.py:94
msgid "Confirm"
msgstr "確認"

#: storage/devicetree.py:95
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""
"本当にデバイス %s のパスフレーズの入力を省略しますか?\n"
"\n"
"このステップをスキップすると、インストール中のデバイス内のコンテンツへのアク"
"セスはできません。"

#: storage/devicetree.py:146
#, fuzzy, python-format
msgid ""
"Error processing drive:\n"
"\n"
"%s\n"
"%-0.fMB\n"
"%s\n"
"\n"
"This device may need to be reinitialized.\n"
"\n"
"REINITIALIZING WILL CAUSE ALL DATA TO BE LOST!%s"
msgstr ""
"ドライブの処理中にエラー発生:\n"
"\n"
"%(path)s\n"
"%(size)-0.fMB\n"
"%(description)s\n"
"\n"
"このデバイスは再初期化が必要かも知れません。\n"
"\n"
"再初期化すると全てのデータが消失します!\n"
"\n"
"このアクションは再初期化を必要とする全ての他のディスクにも適用できるかも知れ"
"ません。%(details)s"

#: storage/devicetree.py:152
#, fuzzy
msgid "_Ignore drive"
msgstr "無視 (_I)"

#: storage/devicetree.py:153
#, fuzzy
msgid "_Re-initialize drive"
msgstr "再初期化 (_R)"

#: storage/devicetree.py:173
#, fuzzy, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %s.  You can reinitialize all related PVs "
"(%s) which will erase the LVM metadata, or ignore which will preserve the "
"contents."
msgstr ""
"LVM をプロセス中にエラー発生。\n"
"%(msg)s 上に一貫性のない LVM データが存在します。LVM メタデータが消失すること"
"になる、全ての関連した PV (%(pvs)s) を再初期化するか、又は、コンテンツを保存"
"することになる無視をすることができます。このアクションは一貫性のないメタデー"
"タを持つ 全ての他の PV にも適用できるかも知れません。"

#: storage/devicetree.py:178
msgid "_Ignore"
msgstr "無視 (_I)"

#: storage/devicetree.py:179
msgid "_Re-initialize"
msgstr "再初期化 (_R)"

#: storage/devicetree.py:1785 storage/devicetree.py:1834
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr "このパーティションは一貫性のない LVM ボリュームグループの一部です。"

#: storage/formats/fs.py:98
msgid "filesystem configuration missing a type"
msgstr "ファイルシステム設定にタイプがありません。"

#: storage/formats/fs.py:328
msgid "Formatting"
msgstr "フォーマット中"

#: storage/formats/fs.py:329
#, fuzzy, python-format
msgid "Creating filesystem on %s"
msgstr "%s 上に %s ファイルシステムを作成"

#: storage/formats/fs.py:428
msgid "Resizing"
msgstr "サイズ変更"

#: storage/formats/fs.py:429
#, python-format
msgid "Resizing filesystem on %s"
msgstr "%s のファイルシステムのサイズ変更"

#: storage/formats/fs.py:472
msgid "Checking"
msgstr "チェック中"

#: storage/formats/fs.py:473
#, python-format
msgid "Checking filesystem on %s"
msgstr "%s のファイルシステムを検査中。"

#: storage/iscsi.py:82 storage/iscsi.py:83
msgid "Scanning iSCSI nodes"
msgstr "iSCSI ノードをスキャン中"

#: storage/iscsi.py:160 storage/iscsi.py:161
msgid "Initializing iSCSI initiator"
msgstr "iSCSI イニシエーターの初期化中"

#: storage/iscsi.py:197
msgid "iSCSI not available"
msgstr "iSCSI は使用できません"

#: storage/iscsi.py:199
msgid "No initiator name set"
msgstr "イニシエーター名がありません"

#: storage/iscsi.py:213
msgid "No iSCSI nodes discovered"
msgstr "iSCSI ノードが見付かりません"

#: storage/iscsi.py:216 storage/iscsi.py:217
msgid "Logging in to iSCSI nodes"
msgstr "iSCSI ノードにログイン中"

#: storage/iscsi.py:239
msgid "No new iSCSI nodes discovered"
msgstr "新しい iSCSI ノードは見付かりません"

#: storage/iscsi.py:242
msgid "Could not log in to any of the discovered nodes"
msgstr "見付かったノードのいずれにもログインできませんでした"

#: storage/partitioning.py:184
msgid ""
"Could not find enough free space for automatic partitioning.  Press 'OK' to "
"exit the installer."
msgstr ""
"自動パーティション設定用に十分な空き領域が見付かりません。インストーラーを抜"
"け出すには'OK'を押してください。"

#: storage/partitioning.py:187
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""
"自動パーティション設定用に十分な空き領域が見付かりません。別の パーティション"
"メソッドを使用して下さい。"

#: storage/partitioning.py:216
msgid "Warnings During Automatic Partitioning"
msgstr "自動パーティション設定中の警告"

#: storage/partitioning.py:218
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"自動パーティション設定中に次の警告が発生しました:\n"
"\n"
"%s"

#: storage/partitioning.py:230 storage/partitioning.py:250
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"「OK」を押すとインストーラを終了します。"

#: storage/partitioning.py:232
#, fuzzy, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%s.%s"
msgstr "要求されたパーティションを設定できませんでした: %s"

#: storage/partitioning.py:252
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""
"\n"
"\n"
"「OK」を押して別のパーティション設定オプションを選択してください。"

#: storage/partitioning.py:254
msgid "Automatic Partitioning Errors"
msgstr "自動パーティション設定のエラーです"

#: storage/partitioning.py:255
#, fuzzy, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %s"
msgstr ""
"パーティション作成に次のエラーが発生しました:\n"
"\n"
"%(errortxt)s\n"
"\n"
"このエラーは、インストール用に十分な領域がハードドライブ上にない場合に発生し"
"ます。%(extra)s"

#: storage/partitioning.py:266
msgid "Unrecoverable Error"
msgstr "修復不能なエラー"

#: storage/zfcp.py:50
msgid "You have not specified a device number or the number is invalid"
msgstr "デバイス番号を指定していないか番号が無効です。"

#: storage/zfcp.py:52
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr "WWPN を指定していないか、そのポート名が無効です。"

#: storage/zfcp.py:54
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr "FCP FUN を指定していないか、番号が無効です。"

#: storage/zfcp.py:132
#, fuzzy, python-format
msgid "Could not free zFCP device %s from device ignore list (%s)."
msgstr ""
"zFCP デバイス %(devnum)s をデバイス無視リスト(%(e)s) から解放できませんでし"
"た。"

#: storage/zfcp.py:137
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr "zFCP デバイス %s が見つからず、デバイス無理リストにもありません。"

#: storage/zfcp.py:150
#, fuzzy, python-format
msgid "Could not set zFCP device %s online (%s)."
msgstr ""
"zFCP デバイス %(devnum)s を オンライン (%(e)s) にセットできませんでした。"

#: storage/zfcp.py:161
#, fuzzy, python-format
msgid "Could not add WWPN %s to zFCP device %s (%s)."
msgstr "WWPN %(wwpn)s を zFCP デバイス %(devnum)s (%(e)s) に追加できません。"

#: storage/zfcp.py:165
#, fuzzy, python-format
msgid "WWPN %s not found at zFCP device %s."
msgstr "WWPN %(wwpn)s が zFCP デバイス %(devnum)s で見つかりません。"

#: storage/zfcp.py:179
#, fuzzy, python-format
msgid "Could not add LUN %s to WWPN %s on zFCP device %s (%s)."
msgstr ""
"LUN %(fcplun)s を zFCP デバイス %(devnum)s (%(e)s)上で WWPN %(wwpn)s に追加で"
"きません、。"

#: storage/zfcp.py:183
#, fuzzy, python-format
msgid "LUN %s at WWPN %s on zFCP device %s already configured."
msgstr ""
"LUN %(fcplun)s は、zFCP デバイス %(devnum)s 上の WWPN %(wwpn)s で既に設定され"
"ています。"

#: storage/zfcp.py:193
#, fuzzy, python-format
msgid ""
"Could not read failed attribute of LUN %s at WWPN %s on zFCP device %s (%s)."
msgstr ""
"zFCP デバイス %(devnum)s (%(e)s) 上の WWPN %(wwpn)s で LUN %(fcplun)s の 失敗"
"属性を読みこむことが できませんでした。"

#: storage/zfcp.py:198
#, fuzzy, python-format
msgid "Failed LUN %s at WWPN %s on zFCP device %s removed again."
msgstr ""
"zFCP デバイス %(devnum)s 上の WWPN %(wwpn)s で失敗した LUN %(fcplun)s が 再度"
"削除されました。"

#: storage/zfcp.py:253
#, fuzzy, python-format
msgid "Could not correctly delete SCSI device of zFCP %s %s %s (%s)."
msgstr ""
"zFCP %(devnum)s %(wwpn)s %(fcplun)s (%(e)s) の SCSI デバイスを正しく消すこと"
"ができませんでした。"

#: storage/zfcp.py:260
#, fuzzy, python-format
msgid "Could not remove LUN %s at WWPN %s on zFCP device %s (%s)."
msgstr ""
"zFCP デバイス %(devnum)s (%(e)s) 上の WWPN %(wwpn)s で LUN %(fcplun)s を削除"
"できませんでした。"

#: storage/zfcp.py:275
#, fuzzy, python-format
msgid "Could not remove WWPN %s on zFCP device %s (%s)."
msgstr ""
"zFCP デバイス %(devnum)s (%(e)s) 上で WWPN %(wwpn)s を削除できませんでした。"

#: storage/zfcp.py:299
#, fuzzy, python-format
msgid "Could not set zFCP device %s offline (%s)."
msgstr "zFCP デバイス %(devnum)s をオフライン (%(e)s) に設定できませんでした。"

#: textw/complete_text.py:32
msgid "Reboot"
msgstr "再起動する"

#: textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr "<Enter> を押して終了します"

#: textw/complete_text.py:62
msgid "Complete"
msgstr "完了"

#: textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr "キーボードの選択"

#: textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr "このコンピューターに接続されるキーボードのモデルを指定してください。"

#: textw/netconfig_text.py:41
#, python-format
msgid "A value is required for the field %s"
msgstr "フィールド %s 用に値が必要です。"

#: textw/netconfig_text.py:86 ui/netconfig.glade.h:9
msgid "Enable network interface"
msgstr "ネットワークインターフェースを有効にする"

#: textw/netconfig_text.py:89 ui/netconfig.glade.h:10
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""
"これを行うにはインストール中にアクティブなネットワーク接続が必要になります。"
"ネットワークインターフェースを設定してください。"

#: textw/netconfig_text.py:128
msgid "Use dynamic IP configuration (DHCP)"
msgstr "動的 IP 設定 (DHCP) を使用"

#: textw/netconfig_text.py:138
msgid "IPv4 Address:"
msgstr "IPv4 アドレス:"

#: textw/netconfig_text.py:161
msgid "Nameserver:"
msgstr "ネームサーバー:"

#: textw/netconfig_text.py:191
msgid "Missing Device"
msgstr "欠如パッケージ"

#: textw/netconfig_text.py:192
msgid "You must select a network device"
msgstr "ネットワークデバイスを選択する必要があります"

#: textw/netconfig_text.py:251
msgid "IPv4 Network Mask "
msgstr "IPv4 ネットワークマスク"

#: textw/netconfig_text.py:274
msgid "Configuring Network Interfaces"
msgstr "ネットワークインターフェースを設定"

#: textw/netconfig_text.py:274
msgid "Waiting for NetworkManager"
msgstr "NetworkManager を待っています。"

#: textw/netconfig_text.py:280
msgid "Error configuring network device"
msgstr "ネットワークデバイス設定中にエラー発生"

#: textw/netconfig_text.py:280
#, python-format
msgid "Error configuring network device %s"
msgstr "ネットワークデバイス %s の設定中にエラー発生"

#: textw/partition_text.py:58
msgid "Partitioning Type"
msgstr "パーティションタイプ"

#: textw/partition_text.py:60
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target."
msgstr ""
"インストールにはハードドライブのパーティション設定が必要です。デフォルトのレ"
"イアウトはほとんどのユーザーに適しています。使用する領域と使用するドライブを "
"インストールのターゲットとして選択します。"

#: textw/partition_text.py:80
msgid "Which drive(s) do you want to use for this installation?"
msgstr "このインストールに使用するドライブを指定してください。"

#: textw/partition_text.py:95
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr "<Space>,<+>,<-> 選択   |   <F2> ドライブの追加   |   <F12> 次の画面"

#: textw/partition_text.py:173 ui/adddrive.glade.h:4
msgid "Advanced Storage Options"
msgstr "高度なストレージオプション"

#: textw/partition_text.py:174 ui/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr "ドライブの構成をどのように変更しますか?"

#: textw/partition_text.py:203
#, fuzzy
msgid "Add FCP Device"
msgstr "FCP デバイスの追加"

#: textw/partition_text.py:204 ui/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""
"zSeries マシンはファイバーチャネル (FCP) 経由で業界基準の SCSI デバイスにアク"
"セスすることができます。それぞれのデバイスに 16 ビットのデバイス番号、64 ビッ"
"トの World Wide Port Name (WWPN) および 64 ビットの FCP LUN を指定する必要が"
"あります。"

#: textw/partition_text.py:223
#, fuzzy
msgid "Add FCoE SAN"
msgstr "_FCoE SAN の追加"

#: textw/partition_text.py:224
msgid ""
"Enter the device name for the NIC which is connected to the FCoE SAN. For "
"example \"eth0\"."
msgstr ""

#: textw/partition_text.py:225
#, fuzzy
msgid "NIC device name"
msgstr "NFS サーバー名:"

#: textw/partition_text.py:232
#, fuzzy, python-format
msgid "%s is not a valid NIC device name."
msgstr "%s は有効な IPv6 アドレスではありません。"

#: textw/partition_text.py:248 ui/iscsi-config.glade.h:7
msgid "Configure iSCSI Parameters"
msgstr "iSCSI パラメーターの設定"

#: textw/partition_text.py:249 ui/iscsi-config.glade.h:8
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""
"iSCSI ディスクを使用するには、ホスト用に設定している iSCSI イニシエーター名及"
"びiSCSI ターゲットのアドレスを入力してください。"

#: textw/partition_text.py:250
#, fuzzy
msgid "Target IP Address"
msgstr "<b>ターゲット IP アドレス(_T):</b>"

#: textw/partition_text.py:251
#, fuzzy
msgid "iSCSI Initiator Name"
msgstr "<b>iSCSI イニシエーター名(_N):</b>"

#: textw/partition_text.py:252
#, fuzzy
msgid "CHAP username"
msgstr "プロキシーのユーザー名 (_S)"

#: textw/partition_text.py:253
#, fuzzy
msgid "CHAP password"
msgstr "パスワード"

#: textw/partition_text.py:254
#, fuzzy
msgid "Reverse CHAP username"
msgstr "<b>逆順 CHAP ユーザー名 (_S):</b>"

#: textw/partition_text.py:255
#, fuzzy
msgid "Reverse CHAP password"
msgstr "<b>逆順 CHAP パスワード (_A):</b>"

#: textw/progress_text.py:46
msgid "Package Installation"
msgstr "パッケージのインストール"

#: textw/timezone_text.py:74
msgid "In which time zone are you located?"
msgstr "どのタイムゾーンに位置しますか?"

#: textw/timezone_text.py:92
msgid "System clock uses UTC"
msgstr "システムクロックに UTC を使用"

#: textw/upgrade_bootloader_text.py:116 textw/upgrade_bootloader_text.py:125
msgid "Update boot loader configuration"
msgstr "ブートローダー設定の更新"

#: textw/upgrade_bootloader_text.py:128
msgid "Skip boot loader updating"
msgstr "ブートローダー更新のスキップ"

#: textw/upgrade_bootloader_text.py:130
msgid "Create new boot loader configuration"
msgstr "新しいブートローダー設定の作成"

#: textw/upgrade_text.py:38
#, fuzzy, python-format
msgid ""
"This release of %s supports the an updated file system, which has several "
"benefits over the file system traditionally shipped in %s.  This "
"installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"%(productName)s のこのリリースでは最新のファイルシステムをサポートしていま"
"す。 これらは従来 %(productName)s に使われていたファイルシステムに比べ優れた"
"点をいくつか備えています。 このインストールプログラムは、データを失うことなく"
"フォーマットしたパーティションを移行することができます。\n"
"\n"
"これらのパーティションのどれを移行しますか?"

#: textw/upgrade_text.py:119
msgid "Free Space"
msgstr "空き領域"

#: textw/upgrade_text.py:137
msgid "RAM detected (MB):"
msgstr "検出された RAM 容量 (MB):"

#: textw/upgrade_text.py:140
msgid "Suggested size (MB):"
msgstr "推奨容量 (MB):"

#: textw/upgrade_text.py:143
msgid "Swap file size (MB):"
msgstr "swap ファイル容量 (MB):"

#: textw/upgrade_text.py:151
msgid "Add Swap"
msgstr "swap の追加"

#: textw/upgrade_text.py:176
msgid "The value you entered is not a valid number."
msgstr "入力した値は無効な数です。"

#: textw/upgrade_text.py:209
msgid "Reinstall System"
msgstr "システムの再インストール"

#: textw/upgrade_text.py:219
msgid "System to Upgrade"
msgstr "アップグレード対象システム"

#: textw/upgrade_text.py:220
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""
"システム上に 既存の Linux インストールがひとつ以上あるようです。\n"
"\n"
"アップグレードするにはひとつを選び、新規にシステムをインストールするには、"
"「システムの再インストール」を選んでください。"

#: textw/userauth_text.py:30
msgid "Root Password"
msgstr "root パスワード"

#: textw/userauth_text.py:33
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""
"ルートパスワードを設定してください。パスワードを2回入力することで、パスワード"
"を覚えていて、タイプミスがないことを確認します。"

#: textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr "root パスワードは 6 文字以上にしてください。"

#: textw/userauth_text.py:84
#, python-format
msgid ""
"You have provided a weak password: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"弱いパスワードが与えられました: %s\n"
"\n"
"このパスワードで続けますか?"

#: textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr "%s"

#: textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"%s へようこそ!\n"
"\n"

#: textw/zipl_text.py:35
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""
"z/IPL ブートローダーはインストールが完了してからシステム上にインストールされ"
"ます。ここで、ご使用のマシンまたは設定が必要とする追加のカーネル及び chandev "
"パラメーターを入力することができます。"

#: textw/zipl_text.py:67
msgid "z/IPL Configuration"
msgstr "z/IPL の設定"

#: textw/zipl_text.py:75 textw/zipl_text.py:79
msgid "Chandev line "
msgstr "Chandev 行"

#: liveinst/liveinst.desktop.in.h:1
msgid "Install"
msgstr "インストール"

#: liveinst/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr "ライブ ディスク をハードディスクにインストール"

#: liveinst/liveinst.desktop.in.h:3
msgid "Install to Hard Drive"
msgstr "ハードドライブにインストール"

#: ui/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""
"このグループに関連付けられるいくつかのパッケージは、インストールする必要があ"
"りません。しかし、インストールすることによって追加機能を提供します。どのパッ"
"ケージをインストールするか選択してください。"

#: ui/GroupSelector.glade.h:2
msgid "_Deselect"
msgstr "選択解除 (_D)"

#: ui/GroupSelector.glade.h:3
msgid "_Deselect all optional packages"
msgstr "すべての追加パッケージの選択を解除 (_D)"

#: ui/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr "追加パッケージ (_O)"

#: ui/GroupSelector.glade.h:5
msgid "_Select"
msgstr "選択 (_S)"

#: ui/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr "すべての追加パッケージを選択 (_S)"

#: ui/GroupSelector.glade.h:7
msgid "dialog1"
msgstr "ダイアログ1"

#: ui/account.glade.h:1
msgid "Confirm:"
msgstr "確認:"

#: ui/account.glade.h:2
msgid "Root Password:"
msgstr "ルートパスワード:"

#: ui/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"root アカウントはシステムの管理に使用されます。 root ユーザーのパスワードを入"
"力してください。"

#: ui/adddrive.glade.h:1
msgid "Add _FCoE SAN"
msgstr "_FCoE SAN の追加"

#: ui/adddrive.glade.h:2
msgid "Add _ZFCP LUN"
msgstr "ZFCP LUN の追加 (_Z)"

#: ui/adddrive.glade.h:3
msgid "Add _iSCSI target"
msgstr "iSCSI ターゲットの追加 (_I)"

#: ui/adddrive.glade.h:6
msgid "_Add drive"
msgstr "ドライブ追加 (_A)"

#: ui/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr "<b>リポジトリー名 (_N):</b>"

#: ui/addrepo.glade.h:2
msgid "<b>Repository _type:</b>"
msgstr "<b>リポジトリータイプ（_T）:</b>"

#: ui/addrepo.glade.h:4
msgid "Configure _proxy"
msgstr "プロキシーの設定 (_P) "

#: ui/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"

#: ui/addrepo.glade.h:9
msgid ""
"Please provide the configuration information for this software repository."
msgstr "このソフトウェアリポジトリーの設定情報を入力してください。"

#: ui/addrepo.glade.h:10
msgid "Proxy U_RL (host:port)"
msgstr "プロキシー U_RL (ホスト:ポート)"

#: ui/addrepo.glade.h:11
msgid "Proxy pass_word"
msgstr "プロキシーのパスワード (_W)"

#: ui/addrepo.glade.h:12
msgid "Proxy u_sername"
msgstr "プロキシーのユーザー名 (_S)"

#: ui/addrepo.glade.h:13
msgid "Repository _URL"
msgstr "リポジトリーの URL (_U)"

#: ui/addrepo.glade.h:14
msgid "Select A Directory"
msgstr "ディレクトリーを選択してください"

#: ui/addrepo.glade.h:15
msgid "URL is a _mirror list"
msgstr "URL は _mirror list です。"

#: ui/addrepo.glade.h:16
msgid "_Directory"
msgstr "ディレクトリー（_D）"

#: ui/addrepo.glade.h:17
msgid "_Options"
msgstr "オプション(_O)"

#: ui/addrepo.glade.h:18
msgid "_Partition"
msgstr "パーティション(_P)"

#: ui/addrepo.glade.h:19
msgid "_Path"
msgstr "パス（_P）"

#: ui/addrepo.glade.h:20
msgid "_Server"
msgstr "サーバー(_S)"

#: ui/anaconda.glade.h:1
msgid "Reboo_t"
msgstr "再起動(_T)"

#: ui/anaconda.glade.h:4
msgid "_Next"
msgstr "次(_N)"

#: ui/autopart.glade.h:1
#, fuzzy
msgid "<b>Shrink _target (in MB):</b>"
msgstr "<b>このサイズ（MB）までパーティションを縮小(_T):</b>"

#: ui/autopart.glade.h:2
#, fuzzy
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target. You can also choose to create your own custom "
"layout."
msgstr ""
"インストールにはハードドライブのパーティション設定が必要です。デフォルトのレ"
"イアウトはほとんどのユーザーに適しています。使用する領域と使用するドライブを "
"インストールのターゲットとして選択します。"

#: ui/autopart.glade.h:3
msgid "Re_view and modify partitioning layout"
msgstr "パーティションレイアウトの再確認と変更(_V)"

#: ui/autopart.glade.h:4
msgid "Volume to Shrink"
msgstr "縮小する量"

#: ui/autopart.glade.h:5
#, fuzzy
msgid "What drive would you like to _boot this installation from?"
msgstr "インストール時に使用する言語を選択してください。"

#: ui/autopart.glade.h:6
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr ""
"インストールするための空きを作るために、どのパーティションを縮小しますか?"

#: ui/autopart.glade.h:7
#, fuzzy
msgid "_Advanced storage configuration"
msgstr "高度なストレージオプション"

#: ui/autopart.glade.h:8
msgid "_Encrypt system"
msgstr "システムを暗号化する (_E)"

#: ui/autopart.glade.h:9
#, fuzzy
msgid "_Select the drive(s) to use for this installation."
msgstr "このインストールに使用するドライブを指定してください。"

#: ui/autopart.glade.h:10
msgid "_Shrink"
msgstr "縮小"

#: ui/blwhere.glade.h:1
msgid "/boot"
msgstr "/boot"

#: ui/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr "BIOS ドライブ順"

#: ui/blwhere.glade.h:3
msgid "Boot loader device"
msgstr "ブートローダーデバイス"

#: ui/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr "第一 BIOS ドライブ:"

#: ui/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr "第四 BIOS ドライブ:"

#: ui/blwhere.glade.h:6
msgid "MBR"
msgstr "MBR"

#: ui/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr "第二 BIOS ドライブ:"

#: ui/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr "第三 BIOS ドライブ:"

#: ui/blwhere.glade.h:9
msgid "Where would you like to install the boot loader for your system?"
msgstr "ブートローダーのインストール場所を指定してください。"

#: ui/detailed-dialog.glade.h:1
msgid "Info"
msgstr "情報"

#: ui/detailed-dialog.glade.h:2
msgid "_Details"
msgstr "詳細 (_D)"

#: ui/fcoe-config.glade.h:1
msgid "Configure FCoE Parameters"
msgstr "FCoE パラメータの設定"

#: ui/fcoe-config.glade.h:2
#, fuzzy
msgid "NIC"
msgstr "NIC:"

#: ui/fcoe-config.glade.h:3
msgid ""
"Please select the network interface which is connected to\n"
"your FCoE switch."
msgstr ""
"使用する FCoE スイッチに接続されているネットワークインターフェイス \n"
"を選択して下さい。"

#: ui/fcoe-config.glade.h:5
msgid "_Add FCoE Disk(s)"
msgstr "FCoE ディスクの追加（_A）"

#: ui/iscsi-config.glade.h:1
msgid "<b>CHAP _Password:</b>"
msgstr "<b>CHAP パスワード (_P):</b>"

#: ui/iscsi-config.glade.h:2
msgid "<b>CHAP _Username:</b>"
msgstr "<b>CHAP ユーザー名 (_U):</b>"

#: ui/iscsi-config.glade.h:3
msgid "<b>Reverse CHAP P_assword:</b>"
msgstr "<b>逆順 CHAP パスワード (_A):</b>"

#: ui/iscsi-config.glade.h:4
msgid "<b>Reverse CHAP U_sername:</b>"
msgstr "<b>逆順 CHAP ユーザー名 (_S):</b>"

#: ui/iscsi-config.glade.h:5
msgid "<b>_Target IP Address:</b>"
msgstr "<b>ターゲット IP アドレス(_T):</b>"

#: ui/iscsi-config.glade.h:6
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr "<b>iSCSI イニシエーター名(_N):</b>"

#: ui/iscsi-config.glade.h:9
msgid "_Add target"
msgstr "ターゲットの追加 (_A)"

#: ui/lukspassphrase.glade.h:1
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""
"暗号化済みの全ての既存デバイスにこのパスフレーズを追加して、ブートプロセスを "
"スムーズにします。"

#: ui/lukspassphrase.glade.h:2
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""
"暗号化されたデバイス用のパスフレーズを選択してください。システムのブート中に"
"選択したパスフレーズの入力を求められます。"

#: ui/lukspassphrase.glade.h:3
msgid "Confirm passphrase:"
msgstr "パスフレーズの確認:"

#: ui/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr "暗号化されたパーティション用のパスフレーズを入力してください。"

#: ui/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr "パスフレーズ:"

#: ui/netconfig.glade.h:2
msgid "<b>Gateway:</b>"
msgstr "<b>ゲートウェイ:</b>"

#: ui/netconfig.glade.h:3
msgid "<b>IPv4 Address:</b>"
msgstr "<b>IPv4 アドレス:</b>"

#: ui/netconfig.glade.h:4
msgid "<b>IPv6 Address:</b>"
msgstr "<b>IPv6 アドレス:</b>"

#: ui/netconfig.glade.h:5
msgid "<b>Nameserver:</b>"
msgstr "<b>ネームサーバー:</b>"

#: ui/netconfig.glade.h:6
msgid "<b>_Interface:</b>"
msgstr "<b>インターフェース (_I):</b>"

#: ui/netconfig.glade.h:7
msgid "Enable IPv_4 support"
msgstr "IPv4 サポートを有効にする (_4)"

#: ui/netconfig.glade.h:8
msgid "Enable IPv_6 support"
msgstr "IPv6 サポートを有効にする (_6)"

#: ui/netconfig.glade.h:11
msgid "Use _dynamic IP configuration (DHCP)"
msgstr "動的 IP 設定 (DHCP) を使用する (_D)"

#: ui/network.glade.h:1
msgid "Hostname:"
msgstr "ホスト名:"

#: ui/network.glade.h:2
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""
"このコンピュータに命名して下さい。ホスト名はネットワーク上でこのコンピュータ"
"を 識別します。"

#: ui/tasksel.glade.h:1
msgid "Customize _later"
msgstr "後でカスタマイズする(_l)"

#: ui/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""
"ソフトウェアのインストールに使用する予定の追加リポジトリーを選択してくださ"
"い。"

#: ui/tasksel.glade.h:4
#, no-c-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""
"%s のデフォルトインストールには一般的なインターネット使用に適用できるソフト"
"ウェアのセットが含まれています。システムにサポートさせる追加タスクを指定して"
"ください。"

#: ui/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""
"ここで、ソフトウェア選択をさらに詳細にカスタマイズすることができます。また、"
"インストール完了後にソフトウェア管理アプリケーションから行うことも可能です。"

#: ui/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr "追加でソフトウェアリポジトリーを加える (_A)"

#: ui/tasksel.glade.h:7
msgid "_Customize now"
msgstr "今すぐカスタマイズする(_C)"

#: ui/tasksel.glade.h:8
msgid "_Modify repository"
msgstr "リポジトリーの編集 (_M)"

#: ui/zfcp-config.glade.h:1
msgid "<b>Device number:</b>"
msgstr "<b>デバイス番号:</b>"

#: ui/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr "<b>FCP LUN:</b>"

#: ui/zfcp-config.glade.h:3
msgid "<b>WWPN:</b>"
msgstr "<b>WWPN:</b>"

#: ui/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr "FCP デバイスの追加"

#: lang-table.h:1
msgid "Afrikaans"
msgstr "アフリカーンス語"

#: lang-table.h:2
msgid "Arabic"
msgstr "アラビア語"

#: lang-table.h:3
msgid "Assamese"
msgstr "アッサム語"

#: lang-table.h:4
msgid "Bengali"
msgstr "ベンガル語"

#: lang-table.h:5
msgid "Bengali(India)"
msgstr "ベンガル語 (インド)"

#: lang-table.h:6
msgid "Bulgarian"
msgstr "ブルガリア語"

#: lang-table.h:7
msgid "Catalan"
msgstr "カタロニア語"

#: lang-table.h:8
msgid "Chinese(Simplified)"
msgstr "中国語 (簡体)"

#: lang-table.h:9
msgid "Chinese(Traditional)"
msgstr "中国語 (繁体)"

#: lang-table.h:10
msgid "Croatian"
msgstr "クロアチア語"

#: lang-table.h:11
msgid "Czech"
msgstr "チェコ語"

#: lang-table.h:12
msgid "Danish"
msgstr "デンマーク語"

#: lang-table.h:13
msgid "Dutch"
msgstr "オランダ語"

#: lang-table.h:14
msgid "English"
msgstr "英語"

#: lang-table.h:15
msgid "Estonian"
msgstr "エストニア語"

#: lang-table.h:16
msgid "Finnish"
msgstr "フィンランド語"

#: lang-table.h:17
msgid "French"
msgstr "フランス語"

#: lang-table.h:18
msgid "German"
msgstr "ドイツ語"

#: lang-table.h:19
msgid "Greek"
msgstr "ギリシャ語"

#: lang-table.h:20
msgid "Gujarati"
msgstr "グジャラート語"

#: lang-table.h:21
msgid "Hebrew"
msgstr "ヘブライ語"

#: lang-table.h:22
msgid "Hindi"
msgstr "ヒンディー語"

#: lang-table.h:23
msgid "Hungarian"
msgstr "ハンガリー語"

#: lang-table.h:24
msgid "Icelandic"
msgstr "アイスランド語"

#: lang-table.h:25
msgid "Iloko"
msgstr "Iloko"

#: lang-table.h:26
msgid "Indonesian"
msgstr "インドネシア語"

#: lang-table.h:27
msgid "Italian"
msgstr "イタリア語"

#: lang-table.h:28
msgid "Japanese"
msgstr "日本語"

#: lang-table.h:29
msgid "Kannada"
msgstr "カンナダ語"

#: lang-table.h:30
msgid "Korean"
msgstr "韓国語"

#: lang-table.h:31
msgid "Macedonian"
msgstr "マケドニア語"

#: lang-table.h:32
msgid "Maithili"
msgstr "マイティリー語"

#: lang-table.h:33
msgid "Malay"
msgstr "マレー語"

#: lang-table.h:34
msgid "Malayalam"
msgstr "マラヤーラム語"

#: lang-table.h:35
msgid "Marathi"
msgstr "マラーティー語"

#: lang-table.h:36
msgid "Nepali"
msgstr "ネパール語"

#: lang-table.h:37
msgid "Norwegian(Bokmål)"
msgstr "ノルウェー語 (Bokmål)"

#: lang-table.h:38
msgid "Northern Sotho"
msgstr "北ソト語"

#: lang-table.h:39
msgid "Oriya"
msgstr "オリヤー語"

#: lang-table.h:40
msgid "Persian"
msgstr "ペルシア語"

#: lang-table.h:41
msgid "Polish"
msgstr "ポーランド語"

#: lang-table.h:42
msgid "Portuguese"
msgstr "ポルトガル語"

#: lang-table.h:43
msgid "Portuguese(Brazilian)"
msgstr "ポルトガル語(ブラジル系)"

#: lang-table.h:44
msgid "Punjabi"
msgstr "パンジャブ語"

#: lang-table.h:45
msgid "Romanian"
msgstr "ルーマニア語"

#: lang-table.h:46
msgid "Russian"
msgstr "ロシア語"

#: lang-table.h:47
msgid "Serbian"
msgstr "セルビア語"

#: lang-table.h:48
msgid "Serbian(Latin)"
msgstr "セルビア語(ラテン系)"

#: lang-table.h:49
msgid "Sinhala"
msgstr "Sinhala"

#: lang-table.h:50
msgid "Slovak"
msgstr "スロバキア語"

#: lang-table.h:51
msgid "Slovenian"
msgstr "スロヴェニア語"

#: lang-table.h:52
msgid "Spanish"
msgstr "スペイン語"

#: lang-table.h:53
msgid "Swedish"
msgstr "スウェーデン語"

#: lang-table.h:54
msgid "Tajik"
msgstr "タジク語"

#: lang-table.h:55
msgid "Tamil"
msgstr "タミル語"

#: lang-table.h:56
msgid "Telugu"
msgstr "テルグ語"

#: lang-table.h:57
msgid "Turkish"
msgstr "トルコ語"

#: lang-table.h:58
msgid "Ukrainian"
msgstr "ウクライナ語"

#: lang-table.h:59
msgid "Vietnamese"
msgstr "ベトナム語"

#: lang-table.h:60
msgid "Welsh"
msgstr "ウェールズ語"

#: lang-table.h:61
msgid "Zulu"
msgstr "ズールー語"

#~ msgid "Ignore _all"
#~ msgstr "全てを無視 (_A)"

#~ msgid "Re-ini_tialize all"
#~ msgstr "全てを再初期化 (_T)"

#~ msgid ""
#~ "Disk %s contains BIOS RAID metadata, but is not part of any recognized "
#~ "BIOS RAID sets. Ignoring disk %s."
#~ msgstr ""
#~ "ディクス %s には、BIOS RAID メタデータが含まれていますが、認識された BIOS "
#~ "RAID セットの一部ではありません。ディスク %s を無視します。"

#~ msgid "Unformatted DASD Device Found"
#~ msgstr "フォーマットしていない DASD デバイスを発見しました"

#~ msgid ""
#~ "Format uninitialized DASD device?\n"
#~ "\n"
#~ "There is %d uninitialized DASD device on this system.  To continue "
#~ "installation, the device must be formatted.  Formatting will remove any "
#~ "data on this device."
#~ msgstr ""
#~ "初期化をしていない DASD デバイスをフォーマットしますか? \n"
#~ "\n"
#~ "このシステムに %d 個の初期化していない DASD デバイスがあります。 インス"
#~ "トールを継続するには、デバイスをフォーマットする必要があります。 フォー"
#~ "マットをするとデバイス上のデータはすべて消滅します。"

#~ msgid ""
#~ "The following error was encountered while downloading the escrow "
#~ "certificate:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "escrow 証明書のダウンロード中に次のエラーに 遭遇しました:\n"
#~ "\n"
#~ "%s"

#~ msgid "%s must have a %s disk label."
#~ msgstr "%s は %s ディスクラベルを持っている必要があります。"

#~ msgid "The passphrase must be at least %d character long."
#~ msgstr "パスフレーズは最低 %d 文字以上でなければなりません。"

#~ msgid "Welcome to %(productName)s for %(productArch)s"
#~ msgstr "%(productArch)s の %(productName)s へようこそ"

#~ msgid "%(productName)s %(productVersion)s installation"
#~ msgstr "%(productName)s %(productVersion)s のインストール"

#~ msgid "Giving up attempting to connect after %d try!\n"
#~ msgstr "%d 回の試行の後で接続試行をあきらめます!\n"

#~ msgid "%s Byte"
#~ msgstr "%s Byte"

#~ msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
#~ msgstr "完了したパッケージ: %(numpkgs)d 個の内 %(donepkgs)d "

#~ msgid ""
#~ "Some of the packages you have selected for install are missing "
#~ "dependencies.  You can exit the installation, go back and change your "
#~ "package selections, or continue installing these packages without their "
#~ "dependencies."
#~ msgstr ""
#~ "インストールするために選択したパッケージの一部は依存関係が欠如していま"
#~ "す。 インストールを終了してパッケージ選択に戻るか、又は依存関係のないパッ"
#~ "ケージの インストールを継続してください。"

#~ msgid ""
#~ "The default installation of %s includes a set of software applicable for "
#~ "general internet usage. You can optionally select a different set of "
#~ "software now."
#~ msgstr ""
#~ "%s のデフォルトインストールには一般的なインターネット使用に適用できるソフ"
#~ "トウェアのセットが含まれています。オプションとして異なる組み合わせのソフト"
#~ "ウェアを 今選択することができます。"

#~ msgid "Graphical Desktop"
#~ msgstr "グラフィカルデスクトップ"

#~ msgid "Minimal"
#~ msgstr "最低限"

#~ msgid ""
#~ "The default installation of %s is a minimum install. You can optionally "
#~ "select a different set of software now."
#~ msgstr ""
#~ "%s のデフォルトインストールは最小限インストールです。オプションとして 異な"
#~ "る組み合わせのソフトウェアを今選択することができます。"

#~ msgid "%(device)s: %(msg)s"
#~ msgstr "%(device)s: %(msg)s"

#~ msgid "%(name)s: %(msg)s"
#~ msgstr "%(name)s: %(msg)s"

#~ msgid "Use All Space"
#~ msgstr "全ての領域を使用する"

#~ msgid ""
#~ "Removes all partitions on the selected device(s).  This includes "
#~ "partitions created by other operating systems.\n"
#~ "\n"
#~ "<b>Tip:</b> This option will remove data from the selected device(s).  "
#~ "Make sure you have backups."
#~ msgstr ""
#~ "選択したデバイス上の全てのパーティションを削除します。これには、他のオペ"
#~ "レーティング システムで作成されたパーティションも含まれます。\n"
#~ "\n"
#~ "<b>ヒント:</b>このオプションは選択したデバイスからデータを削除します。 "
#~ "バックアップがあることを確認してください。"

#~ msgid "Replace Existing Linux System(s)"
#~ msgstr "既存の Linux システムを入れ替える"

#~ msgid ""
#~ "Removes only Linux partitions (created from a previous Linux "
#~ "installation).  This does not remove other partitions you may have on "
#~ "your storage device(s) (such as VFAT or FAT32).\n"
#~ "\n"
#~ "<b>Tip:</b> This option will remove data from the selected device(s).  "
#~ "Make sure you have backups."
#~ msgstr ""
#~ "Linux のパーティションのみを削除します（以前の Linux インストールで作成し"
#~ "たもの）。 これは、ストレージデバイス上に存在する他のパーティションは削除"
#~ "しません （VFAT や FAT32 など）。\n"
#~ "\n"
#~ "<b>ヒント:</b> このオプションは、選択したデバイスからデータを削除します。"
#~ "バックアップが あることを確認して下さい。"

#~ msgid ""
#~ "Shrinks existing partitions to create free space for the default layout."
#~ msgstr ""
#~ "現在のパーティションを縮小してデフォルトレイアウト用に空き領域を作成しま"
#~ "す。"

#~ msgid "Use Free Space"
#~ msgstr "空き領域を使用する"

#~ msgid ""
#~ "Retains your current data and partitions and uses only the unpartitioned "
#~ "space on the selected device(s), assuming you have enough free space "
#~ "available."
#~ msgstr ""
#~ "現在のデータとパーティションを維持して、選択したデバイスに十分な空き領域が"
#~ "ある限りは そのパーティション未設定領域のみを使用します。"

#~ msgid ""
#~ "Manually create your own custom layout on the selected device(s) using "
#~ "our partitioning tool."
#~ msgstr ""
#~ "パーティション設定ツールを使用して選択したデバイス上に手動で個人設定のカス"
#~ "タムレイアウトを 作成します。"

#~ msgid "Clear Disks Selector"
#~ msgstr "ディスクセレクタを空にする"

#~ msgid "You must select at least one drive to be used for installation."
#~ msgstr ""
#~ "インストール用に少なくともハードドライブを 1 つ選択しなければなりません。"

#~ msgid "You must select one drive to boot from."
#~ msgstr "ブート元のドライブをを１つ選択する必要があります。"

#~ msgid "Model"
#~ msgstr "モデル"

#~ msgid "Capacity"
#~ msgstr "容量"

#~ msgid "Vendor"
#~ msgstr "ベンダー"

#~ msgid "Interconnect"
#~ msgstr "相互接続"

#~ msgid "Serial Number"
#~ msgstr "シリアル番号"

#~ msgid "Boot"
#~ msgstr "ブート"

#~ msgid ""
#~ "<b>Tip:</b> All Linux filesystems on install target devices will be "
#~ "reformatted and wiped of any data.  Make sure you have backups."
#~ msgstr ""
#~ "<b>ヒント:</b> インストール先のデバイス上にある全ての Linux ファイルシステ"
#~ "ムは 再フォーマットされてデータは抹消されます。バックアップがあることを確"
#~ "認して下さい。"

#~ msgid ""
#~ "<b>Tip:</b> Install target devices will be reformatted and wiped of any "
#~ "data.  Make sure you have backups."
#~ msgstr ""
#~ "<b>ヒント:</b> インストール先のデバイスは再フォーマットされてデータは抹消"
#~ "されます。 バックアップがあることを確認して下さい。"

#~ msgid ""
#~ "<b>Tip:</b> Your filesystems on install target devices will not be wiped "
#~ "unless you choose to do so during customization."
#~ msgstr ""
#~ "<b>ヒント:</b> インストール先のデバイス上にあるファイルシステムは、カスタ"
#~ "マイズ中に 削除の選択をしていない限りは抹消されません。"

#~ msgid "Fresh Installation"
#~ msgstr "新規インストール"

#~ msgid ""
#~ "At least one existing installation has been detected on your system.  "
#~ "What would you like to do?"
#~ msgstr ""
#~ "使用システム上に少なくとも１つの既存インストールが検出されました。どうした"
#~ "いですか?"

#~ msgid "<b>Which %s installation would you like to upgrade?</b>"
#~ msgstr "<b>どの %s インストールをアップグレードしたいですか?</b>"

#~ msgid ""
#~ "<b>%s device(s) (%s MB) selected</b> out of %s device(s) (%s MB) total."
#~ msgstr "%s device(s) 合計 (%s MB) の内 <b>%s device(s) (%s MB) を選択</b>"

#~ msgid "Device Filter"
#~ msgstr "デバイスフィルター"

#~ msgid "Identifier"
#~ msgstr "識別子"

#~ msgid "Paths"
#~ msgstr "パス"

#~ msgid "Port"
#~ msgstr "ポート"

#~ msgid "Target"
#~ msgstr "ターゲット"

#~ msgid "LUN"
#~ msgstr "LUN"

#~ msgid "What type of devices will your installation involve?"
#~ msgstr "インストールはどんなタイプのデバイスを使用しますか?"

#~ msgid "Basic Storage Devices"
#~ msgstr "基本ストレージデバイス"

#~ msgid ""
#~ "Installs or upgrades to typical types of storage devices.  If you're not "
#~ "sure which option is right for you, this is probably it."
#~ msgstr ""
#~ "標準タイプのストレージデバイスにインストール、又はアップグレードします。 "
#~ "どのオプションが正しいのか不明な場合は、これが適切でしょう。"

#~ msgid "Specialized Storage Devices"
#~ msgstr "特殊化したストレージデバイス"

#~ msgid ""
#~ "Installs or upgrades to devices such as Storage Area Networks (SANs) or "
#~ "mainframe attached disks (DASD), usually in an enterprise environment"
#~ msgstr ""
#~ "SAN (Storage Area Network)、あるいはメインフレームに接続されたディスク "
#~ "(DASD) など 通常はエンタープライズ環境にあるデバイスにインストール、又は"
#~ "アップグレードします。"

#~ msgid "You cannot create more than %d logical volume per volume group."
#~ msgstr "ボリュームグループ毎に %d 以上の論理ボリュームは作成できません。"

#~ msgid "LVM Volume Group %s (%-0.f MB)"
#~ msgstr "LVM ボリュームグループ %s (%-0.f MB)"

#~ msgid "MD RAID ARRAY %s (%-0.f MB)"
#~ msgstr "MD RAID アレイ %s (%-0.f MB)"

#~ msgid ""
#~ "The following pre-existing devices have been selected to be formatted, "
#~ "destroying all data."
#~ msgstr ""
#~ "次の既存のデバイスをフォーマットするよう選択されています。これによりすべて"
#~ "のデータが消去されます。"

#~ msgid "Cannot perform any creation action"
#~ msgstr "作成作業を実行できません"

#~ msgid ""
#~ "Note that the creation action requires one of the following:\n"
#~ "\n"
#~ "* Free space in one of the Hard Drives.\n"
#~ "* At least two free Software RAID partitions.\n"
#~ "* At least one free physical volume (LVM) partition.\n"
#~ "* At least one Volume Group with free space."
#~ msgstr ""
#~ "作成作業には以下のいずれかが必要なことに注意して下さい:\n"
#~ "\n"
#~ "* ハードドライブの１つに空き領域。\n"
#~ "* ソフトウェア RAID に最低 2つの空きパーティション。 \n"
#~ "* 物理ボリューム (LVM) に最低１つの空きパーティション。\n"
#~ "* 最低１つの空き領域を持つボリュームグループ。"

#~ msgid "You currently have %d software RAID partition free to use."
#~ msgstr "%d 個のソフトウェア RAID パーティションの空き領域が使用できます。"

#~ msgid "About RAID"
#~ msgstr "RAID について"

#~ msgid ""
#~ "Logical Volume Manager (LVM) is a 3 level construct. The fist level is "
#~ "made up of disks or partitions formated with LVM metadata called Physical "
#~ "Volumes (PV).  A Volume Group (VG) sits on top of one or more PVs. The "
#~ "VG, in turn, is the base to creat one ore more Logical Volumes (LV).  "
#~ "Note that a VG can be an aggregate of PVs from multiple physical disk.  "
#~ "For more information on using LVM please consult the %s documentation\n"
#~ msgstr ""
#~ "LVM (Logical Volume Manager) は３レベル構造になっています。最初のレベルは "
#~ "LVM メタデータでフォーマットされた物理ボリューム (PV) と呼ばれるディスク、"
#~ "又は パーティションの集合です。ボリュームグループ (VG) は単数、又は複数の "
#~ "PV から構成 されます。そして次には VG が単数、又は複数の論理ボリューム "
#~ "(LV) を構成する土台と なります。VG は複数の物理ディスクから出来る PV の総"
#~ "合体であることに留意して 下さい。LVM の使用法についての詳細には、%s ドキュ"
#~ "メントをご覧下さい。\n"

#~ msgid ""
#~ "To create a PV you need a partition with free space.  To create a VG you "
#~ "need a PV that is not part of any existing VG.  To create a LV you need a "
#~ "VG with free space.\n"
#~ "\n"
#~ msgstr ""
#~ "PV を作成するには、空き領域を持つパーティションが必要です。VG を作成するに"
#~ "は、 既存の VG の一部ではない PV が必要です。LV を作成するには、空き領域を"
#~ "持つ VG が必要になります。\n"
#~ "\n"

#~ msgid "You currently have %d available PV free to use.\n"
#~ msgstr "現在、%d 個の PV の空き領域が使用できます。\n"

#~ msgid "You currently have free space to create PVs."
#~ msgstr "現在、PV を作成するための空き領域があります。"

#~ msgid "About LVM"
#~ msgstr "LVM について"

#~ msgid "_Create"
#~ msgstr "作成（_C）"

#~ msgid "Please Select A Device"
#~ msgstr "デバイスを １つ選択してください。"

#~ msgid "Clone Drive Tool"
#~ msgstr "ドライブのクローンツール"

#~ msgid ""
#~ "This tool clones the layout from a partitioned source onto other similar "
#~ "sized drives. The source must have partitions which are restricted to "
#~ "that drive and must ONLY contain unused software RAID partitions.  "
#~ "EVERYTHING on the target drive(s) will be destroyed.\n"
#~ msgstr ""
#~ "このツールはパーティション設定を持つソースからのレイアウトを他の同様な サ"
#~ "イズのドライブにクローンします。このソース（クローン元）はこのドライブに制"
#~ "約された パーティションを持っている必要があり、未使用のソフトウェア RAID "
#~ "パーティションのみ を含んでいる必要があります。ターゲットドライブ（クロー"
#~ "ン先）上の全ては消去されます。\n"

#~ msgid "Missing options on line %d of kickstart file %s."
#~ msgstr "%d 行目のオプションがキックスタートファイル %s にありません。"

#~ msgid "NFS mount options (optional):"
#~ msgstr "NFS マウントポイント（オプション）"

#~ msgid "Enable HTTP proxy"
#~ msgstr "HTTP プロキシを有効にする"

#~ msgid "Proxy URL"
#~ msgstr "プロキシ URL"

#~ msgid "Username"
#~ msgstr "ユーザー名"

#~ msgid "Formatting DASD Device"
#~ msgstr "DASD デバイスのフォーマット"

#~ msgid "Preparing %d DASD device for use with Linux..."
#~ msgstr "Linux での使用の為に %d DASD デバイスを準備中..."

#~ msgid "Creating"
#~ msgstr "作成"

#~ msgid "Creating device %s"
#~ msgstr "デバイス %s の作成 "

#~ msgid "A RAID%d set requires atleast %d members"
#~ msgstr "RAID%d セットには少なくとも %d 個のメンバーが必要です。"

#~ msgid "Creating file %s"
#~ msgstr "ファイル %s の作成"

#~ msgid "Connecting to FCoE SAN"
#~ msgstr "FCoE SAN へ接続"

#~ msgid "FCoE not available"
#~ msgstr "FCoE は使用できません"

#~ msgid "Unknown return code: %d."
#~ msgstr "不明な戻しコードです: %d"

#~ msgid "%(type)s filesystem check failure on %(device)s: "
#~ msgstr "%(device)s 上で %(type)s ファイルシステムのチェックに失敗: "

#~ msgid ""
#~ "Errors like this usually mean there is a problem with the filesystem that "
#~ "will require user interaction to repair.  Before restarting installation, "
#~ "reboot to rescue mode or another system that allows you to repair the "
#~ "filesystem interactively.  Restart installation after you have corrected "
#~ "the problems on the filesystem."
#~ msgstr ""
#~ "このようなエラーは通常、ファイルシステムに問題があることを意味しており、 "
#~ "その修正にはユーザーの介入を必要とします。インストールを再開始する前に レ"
#~ "スキューモードで、又はファイルシステムを対話式で修正できる他のシステムで "
#~ "再起動して下さい。ファイルシステムの問題を修正してからインストールを再開"
#~ "始 して下さい。"

#~ msgid "File system errors left uncorrected."
#~ msgstr "ファイルシステムエラーが未修正のままです。"

#~ msgid "Operational error."
#~ msgstr "操作上のエラー"

#~ msgid "Usage or syntax error."
#~ msgstr "ユーザー又は構文のエラー"

#~ msgid "e2fsck cancelled by user request."
#~ msgstr "ユーザーの指示で e2fsck がキャンセルされました。"

#~ msgid "Shared library error."
#~ msgstr "共有ライブラリのエラー"

#~ msgid ""
#~ "Recoverable errors have been detected or dosfsck has discovered an "
#~ "internal inconsistency."
#~ msgstr ""
#~ "復元可能なエラーは検出されたか、又は dosfsck が内部の整合性問題を発見しま"
#~ "した。"

#~ msgid "Usage error."
#~ msgstr "使用法エラー"

#~ msgid "Encrypted"
#~ msgstr "暗号化"

#~ msgid "Encrypting %s"
#~ msgstr "暗号化 %s"

#~ msgid "Creating %s on %s"
#~ msgstr "%s 上に %s を作成"

#~ msgid ""
#~ "Could not allocate requested partitions: \n"
#~ "\n"
#~ "%(msg)s.%(extra)s"
#~ msgstr ""
#~ "要求されたパーティションを割り当てできませんでした: \n"
#~ "\n"
#~ "%(msg)s.%(extra)s"

#~ msgid "Which type of installation would you like?"
#~ msgstr "どのタイプのインストールをしますか ?"

#~ msgid "<b>Data Storage Devices</b> (to be mounted only)"
#~ msgstr "<b>データストレージデバイス</b> (マウントのみ)"

#~ msgid "<b>Install Target Devices</b>"
#~ msgstr "<b>インストール先のデバイス</b>"

#~ msgid ""
#~ "Below are the storage devices you've selected to be a part of this "
#~ "installation.  Please indicate using the arrows below which devices you'd "
#~ "like to use as data drives (these will not be formatted, only mounted) "
#~ "and which devices you'd like to use as system drives (these may be "
#~ "formatted)."
#~ msgstr ""
#~ "このインストールの一部として選択したストレージデバイスを以下に示します。 "
#~ "以下の矢印を使用してデータドライブとして使用したいデバイス（これらはフォー"
#~ "マットされずにマウントされるだけ）と システムドライブとして使用したいデバ"
#~ "イス（これらはフォーマットできます）を示して下さい。 "

#~ msgid ""
#~ "<span size=\"small\" color=\"gray\">Create a RAID formated partition</"
#~ "span>"
#~ msgstr ""
#~ "<span size=\"small\" color=\"gray\">RAID フォーマットをしたパーティション"
#~ "を作成</span>"

#~ msgid ""
#~ "<span size=\"small\" color=\"gray\">Create a logical volume on selected "
#~ "volume group</span>"
#~ msgstr ""
#~ "<span size=\"small\" color=\"gray\">選択したボリュームグループ上に論理ボ"
#~ "リュームを作成 </span>"

#~ msgid ""
#~ "<span size=\"small\" color=\"gray\">Create an LVM formated partition</"
#~ "span>"
#~ msgstr ""
#~ "<span size=\"small\" color=\"gray\">LVM フォーマットしたパーティションを作"
#~ "成</span>"

#~ msgid ""
#~ "<span size=\"small\" color=\"gray\">Facilitates RAID device creation</"
#~ "span>"
#~ msgstr ""
#~ "<span size=\"small\" color=\"gray\">RAID デバイスの作成を用意</span>"

#~ msgid ""
#~ "<span size=\"small\" color=\"gray\">General purpose partition creation</"
#~ "span>"
#~ msgstr ""
#~ "<span size=\"small\" color=\"gray\">汎用目的のパーティションを作成</span>"

#~ msgid ""
#~ "<span size=\"small\" color=\"gray\">Requires at least 1 free LVM formated "
#~ "partition</span>"
#~ msgstr ""
#~ "<span size=\"small\" color=\"gray\">LVM フォーマットの空きパーティションが"
#~ "最低でも１つ必要 </span>"

#~ msgid ""
#~ "<span size=\"small\" color=\"gray\">Requires at least 2 free RAID "
#~ "formated partitions</span>"
#~ msgstr ""
#~ "<span size=\"small\" color=\"gray\">RAID フォーマットの空きパーティション"
#~ "が最低でも２つ必要</span>"

#~ msgid "Create"
#~ msgstr "作成"

#~ msgid "Create LVM"
#~ msgstr "LVM を作成"

#~ msgid "Create Partition"
#~ msgstr "パーティションを作成"

#~ msgid "Create Storage"
#~ msgstr "ストレージを作成"

#~ msgid "LVM Logical Volume"
#~ msgstr "LVM 論理ボリューム"

#~ msgid "LVM Physical Volume"
#~ msgstr "LVM 物理ボリューム"

#~ msgid "LVM Volume Group"
#~ msgstr "LVM ボリュームグループ"

#~ msgid "RAID Clone"
#~ msgstr "RAID クローン"

#~ msgid "RAID Partition"
#~ msgstr "RAID パーティション"

#~ msgid "Standard Partition"
#~ msgstr "標準パーティション"

#~ msgid "Use DCB"
#~ msgstr "DCB の使用"

#~ msgid "<b>%s devices (%s) selected</b> out of %s devices (%s) total."
#~ msgstr "%s デバイスの合計 (%s) の内 <b>%s デバイス の (%s) を選択</b>"

#~ msgid ""
#~ "<b>Tip:</b> Selecting a drive on this screen does not necessarily mean it "
#~ "will be wiped by the installation process.  Also, note that post-"
#~ "installation you may mount drives you did not select here by modifying "
#~ "your /etc/fstab file."
#~ msgstr ""
#~ "<b>ヒント:</b> この画面でドライブを選択しても必ずしもインストールプロセス"
#~ "によって 消去されることにはなりません。また、インストール後に /etc/fstab "
#~ "ファイルを編集することで ここで選択していないドライブをマウントすることも"
#~ "出来ます。 "

#~ msgid "Add Advanced Target"
#~ msgstr "高度なターゲットの追加"

#~ msgid "Basic Devices"
#~ msgstr "基本デバイス"

#~ msgid "Filter By:"
#~ msgstr "フィルター項目:"

#~ msgid "Firmware RAID"
#~ msgstr "ファームウェア RAID"

#~ msgid "Identifier:"
#~ msgstr "識別子:"

#~ msgid ""
#~ "Interconnect\n"
#~ "Vendor\n"
#~ "Identifier"
#~ msgstr ""
#~ "相互接続\n"
#~ "ベンダー\n"
#~ "識別子"

#~ msgid "LUN:"
#~ msgstr "LUN:"

#~ msgid "Multipath Devices"
#~ msgstr "マルチパスデバイス"

#~ msgid "Other SAN Devices"
#~ msgstr "他の SAN デバイス"

#~ msgid ""
#~ "Please select the drives you'd like to install the operating system on, "
#~ "as well as any drives you'd like to automatically mount to your system, "
#~ "below:"
#~ msgstr ""
#~ "オペレーティングシステムのインストール先となるデバイスを選択して下さい。更"
#~ "には システムに自動的にマウントを希望するいずれかのドライブも以下で選択し"
#~ "てください:"

#~ msgid ""
#~ "Port / Target / LUN\n"
#~ "Target Identifier"
#~ msgstr ""
#~ "ポート / ターゲット / LUN\n"
#~ "ターゲット識別子"

#~ msgid "Port:"
#~ msgstr "ポート:"

#~ msgid "Search"
#~ msgstr "検索"

#~ msgid "Search By:"
#~ msgstr "検索項目:"

#~ msgid "Search Results:"
#~ msgstr "検索結果:"

#~ msgid "Show Identifiers that Include:"
#~ msgstr "以下を含む識別子を表示します:"

#~ msgid "Show Only Devices From:"
#~ msgstr "以下からのデバイスのみを表示します:"

#~ msgid "Show Only Devices Using:"
#~ msgstr "以下を使用したデバイスのみを表示します:"

#~ msgid "Target:"
#~ msgstr "ターゲット:"

#~ msgid "Configure Network"
#~ msgstr "ネットワークの設定"
